<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
  <head>
    <!-- Google Tag Manager -->
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-MR59FCR');</script>
    <!-- End Google Tag Manager -->
    <script type="text/javascript" src="/js/config.js"></script>
    <meta http-equiv="cache-control" content="max-age=0" />
    <meta http-equiv="cache-control" content="no-cache" />
    <meta http-equiv="expires" content="0" />
    <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
    <meta http-equiv="pragma" content="no-cache" />
    <link rel="canonical" href="https://www.aging-us.com/article/206311/text" />
    <meta data-meta-name="citation_pdf_url" content="https://www.aging-us.com/article/206311/pdf" name="citation_pdf_url">
<meta data-meta-name="citation_keywords" content="aging, dementia, SRC-1, S100A6" name="citation_keywords">
<meta data-meta-name="keywords" content="aging, dementia, SRC-1, S100A6" name="keywords">
<meta data-meta-name="citation_abstract_html_url" content="https://www.aging-us.com/article/206311" name="citation_abstract_html_url">
<meta data-meta-name="citation_publication_date" content="2025-08-27" name="citation_publication_date">
<meta data-meta-name="citation_date" content="2025-08-27" name="citation_date">
<meta data-meta-name="citation_lastpage" name="citation_lastpage" content="2188">
<meta data-meta-name="citation_firstpage" name="citation_firstpage" content="2167">
<meta data-meta-name="citation_issue" name="citation_issue" content="8">
<meta data-meta-name="citation_volume" name="citation_volume" content="17">
<meta data-meta-name="citation_pmid" content="40882002" name="citation_pmid">
<meta data-meta-name="citation_doi" content="10.18632/aging.206311" name="citation_doi">
<meta data-meta-name="citation_title" content="A protective role of SRC-1 against aging associated cognitive decline" name="citation_title">
<meta data-meta-name="citation_journal_title" content="Aging" name="citation_journal_title">
<meta data-meta-name="citation_issn" content="1945-4589" name="citation_issn">
<meta data-meta-name="author-1" name="citation_author" content="Hesong Liu">
<meta data-meta-name="author-2" name="citation_author" content="Yongjie Yang">
<meta data-meta-name="author-3" name="citation_author" content="Jonathan C. Bean">
<meta data-meta-name="author-4" name="citation_author" content="Yang He">
<meta data-meta-name="author-5" name="citation_author" content="Hailan Liu">
<meta data-meta-name="author-6" name="citation_author" content="Rambabu Majji">
<meta data-meta-name="author-7" name="citation_author" content="Chen Liang">
<meta data-meta-name="author-8" name="citation_author" content="Nan Zhang">
<meta data-meta-name="author-9" name="citation_author" content="Meng Yu">
<meta data-meta-name="author-10" name="citation_author" content="Longlong Tu">
<meta data-meta-name="author-11" name="citation_author" content="Qingzhuo Liu">
<meta data-meta-name="author-12" name="citation_author" content="Yue Deng">
<meta data-meta-name="author-13" name="citation_author" content="Kristine M. Conde">
<meta data-meta-name="author-14" name="citation_author" content="Na Yin">
<meta data-meta-name="author-15" name="citation_author" content="Mengjie Wang">
<meta data-meta-name="author-16" name="citation_author" content="Yongxiang Li">
<meta data-meta-name="author-17" name="citation_author" content="Junying Han">
<meta data-meta-name="author-18" name="citation_author" content="Sanika Vattakuzhiyil Jossy">
<meta data-meta-name="author-19" name="citation_author" content="Megan Elyse Burt">
<meta data-meta-name="author-20" name="citation_author" content="Hari Krishna Yalamanchili">
<meta data-meta-name="author-21" name="citation_author" content="Chunmei Wang">
<meta data-meta-name="twitter:site" content="AgingJrnl" name="twitter:site">
<meta data-meta-name="twitter:card" content="summary_large_image" name="twitter:card">
<meta data-meta-name="twitter:image" content="https://www.aging-us.com/article/206311/figure/f1/large" name="twitter:image">
<meta data-meta-name="og:image" content="https://www.aging-us.com/article/206311/figure/f1/large" property="og:image">
<meta data-meta-name="og:description" property="og:description" content="Aging | doi:10.18632/aging.206311. Hesong Liu, Yongjie Yang, Jonathan C. Bean, Yang He, Hailan Liu, Rambabu Majji, Chen Liang, Nan Zhang, Meng Yu, Longlong Tu, Qingzhuo Liu, Yue Deng, Kristine M. Conde, Na Yin, Mengjie Wang, Yongxiang Li, Junying Han, Sanika Vattakuzhiyil Jossy, Megan Elyse Burt, Hari Krishna Yalamanchili, Chunmei Wang">
<meta data-meta-name="twitter:description" name="twitter:description" content="Aging | doi:10.18632/aging.206311. Hesong Liu, Yongjie Yang, Jonathan C. Bean, Yang He, Hailan Liu, Rambabu Majji, Chen Liang, Nan Zhang, Meng Yu, Longlong Tu, Qingzhuo Liu, Yue Deng, Kristine M. Conde, Na Yin, Mengjie Wang, Yongxiang Li, Junying Han, Sanika Vattakuzhiyil Jossy, Megan Elyse Burt, Hari Krishna Yalamanchili, Chunmei Wang">
<meta data-meta-name="description" content="Introduction: Research indicates a strong correlation between obesity and the risk of dementia, both are linked to steroid receptor coactivator-1 (SRC-1), a transcriptional coactivator.
Methods: We used RNA sequencing analysis (RNA-Seq) to investigate the transcriptome of SRC-1-KO mice, and identified S100 calcium-binding protein A6 (S100A6), an AD associated gene, as one target of SRC-1. We tested cognitive behaviors in SRC-1-KO mice and mice with a humanized SRC-1 mutation (SRC-1L1376P), and performed promoter luciferase assays on S100A6.
Results: Loss of SRC-1 caused alterations in gene signatures that are commonly associated with neurodegenerative diseases, including AD, and diminished the neural plasticity of the hippocampal CA1 neurons. Both SRC-1-KO and SRC-1L1376P mice displayed early signs of contextual memory impairment at 6 months of age. Mechanistically, SRC-1 significantly promoted the expression S100A6.
Conclusion: We identified a protective role of SRC1 against aging associated cognitive decline, potentially by promoting the expression of S100A6." name="description">
<meta data-meta-name="og:url" itemprop="url" property="og:url" content="https://www.aging-us.com/article/206311/text">
<meta data-meta-name="og:type" property="og:type" content="website">
<meta data-meta-name="twitter:title" name="twitter:title" content="A protective role of SRC-1 against aging associated cognitive decline | Aging">
<meta data-meta-name="og:title" property="og:title" content="A protective role of SRC-1 against aging associated cognitive decline | Aging">
<meta data-meta-name="title" content="A protective role of SRC-1 against aging associated cognitive decline | Aging" name="title">

<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.4.2/css/all.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/brands.min.css">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
<link rel="stylesheet" type="text/css" href="/aging-us-ba6298b/css/aging.css">
<link href="//cdn.jsdelivr.net/npm/featherlight@1.7.14/release/featherlight.min.css" type="text/css" rel="stylesheet" />

<script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js"></script>
<script src="//cdn.jsdelivr.net/npm/featherlight@1.7.14/release/featherlight.min.js" type="text/javascript" charset="utf-8"></script>
<script src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-dae2a658e53c442cd84f4390def938db5569f7911cfa1aa78d8970054ec2f48f.js"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/mml-chtml.js"></script>
<script type="text/javascript" src="/aging-us-ba6298b/js/static-site.js"></script>
<script type="application/ld+json">
{
 "@context": "http://schema.org",
 "@graph": [
   {
       "@type": "MedicalScholarlyArticle",
       "headline": "A protective role of SRC-1 against aging associated cognitive decline",
       "description": "Introduction: Research indicates a strong correlation between obesity and the risk of dementia, both are linked to steroid receptor coactivator-1 (SRC-1), a transcriptional coactivator.
Methods: We used RNA sequencing analysis (RNA-Seq) to investigate the transcriptome of SRC-1-KO mice, and identified S100 calcium-binding protein A6 (S100A6), an AD associated gene, as one target of SRC-1. We tested cognitive behaviors in SRC-1-KO mice and mice with a humanized SRC-1 mutation (SRC-1L1376P), and performed promoter luciferase assays on S100A6.
Results: Loss of SRC-1 caused alterations in gene signatures that are commonly associated with neurodegenerative diseases, including AD, and diminished the neural plasticity of the hippocampal CA1 neurons. Both SRC-1-KO and SRC-1L1376P mice displayed early signs of contextual memory impairment at 6 months of age. Mechanistically, SRC-1 significantly promoted the expression S100A6.
Conclusion: We identified a protective role of SRC1 against aging associated cognitive decline, potentially by promoting the expression of S100A6.",
       "sameAs": "https://doi.org/10.18632/aging.206311",
       "image": "https://www.aging-us.com/article/206311/figure/f1/large",
       "datePublished": "2025-08-27T00:00:00Z",
       "mainEntityOfPage": {
           "@type": "WebPage",
           "@id": "https://www.aging-us.com/article/206311/text",
           "publisher": {
               "@type": "Organization",
               "name": "Aging-US",
               "logo": {
                   "@type": "ImageObject",
                   "url": "https://www.aging-us.com/images/aging-logo.png",
                   "width": 163,
                   "height": 74
               }
           }
       },
       "author": [
 {
 "@type": "Person",
           "name": "Liu, Hesong"
       },
 {
 "@type": "Person",
           "name": "Yang, Yongjie"
       },
 {
 "@type": "Person",
           "name": "Bean, Jonathan C."
       },
 {
 "@type": "Person",
           "name": "He, Yang"
       },
 {
 "@type": "Person",
           "name": "Liu, Hailan"
       },
 {
 "@type": "Person",
           "name": "Majji, Rambabu"
       },
 {
 "@type": "Person",
           "name": "Liang, Chen"
       },
 {
 "@type": "Person",
           "name": "Zhang, Nan"
       },
 {
 "@type": "Person",
           "name": "Yu, Meng"
       },
 {
 "@type": "Person",
           "name": "Tu, Longlong"
       },
 {
 "@type": "Person",
           "name": "Liu, Qingzhuo"
       },
 {
 "@type": "Person",
           "name": "Deng, Yue"
       },
 {
 "@type": "Person",
           "name": "Conde, Kristine M."
       },
 {
 "@type": "Person",
           "name": "Yin, Na"
       },
 {
 "@type": "Person",
           "name": "Wang, Mengjie"
       },
 {
 "@type": "Person",
           "name": "Li, Yongxiang"
       },
 {
 "@type": "Person",
           "name": "Han, Junying"
       },
 {
 "@type": "Person",
           "name": "Jossy, Sanika Vattakuzhiyil"
       },
 {
 "@type": "Person",
           "name": "Burt, Megan Elyse"
       },
 {
 "@type": "Person",
           "name": "Yalamanchili, Hari Krishna"
       },
 {
 "@type": "Person",
           "name": "Wang, Chunmei"
       }
   ]}
 ]
}
</script>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-74807910-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

gtag('config', 'UA-74807910-2');
gtag('config', 'G-839950QGRH');
</script>

<meta name="fragment" content="!">
<title>A protective role of SRC-1 against aging associated cognitive decline | Aging</title>


  </head>
  <body>
    <!-- Google Tag Manager (noscript) -->
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MR59FCR"
                      height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <!-- End Google Tag Manager (noscript) -->
    <!-- ========= Navigation ========= -->
    <header class="navigation">
      <div style="display:flex; align-items:center; justify-content:space-between;">

        <div class="nav-sidebar">
          <a href="/"><img src="/images/aging-logo.png" class="logo left" alt="Aging">
            <span class="journal-name-extra white-text left"></span>
          </a>
        </div>

        <div id="top-social-links-desktop-container" style=" margin-right:45px;">
          <div id="top-social-etal">
            <ul id="top-social-links">
                  <li class="social-links">
                    <a href="https://www.facebook.com/Aging-218534691828805/" target="_blank">
                      <span class="fa-brands fa-facebook"></span>
                      
                      
                    </a>
                  </li>
                  <li class="social-links">
                    <a href="https://twitter.com/AgingJrnl" target="_blank">
                      <span class="fa-brands fa-x-twitter"></span>
                      
                      
                    </a>
                  </li>
                  <li class="social-links">
                    <a href="https://www.instagram.com/agingjrnl/" target="_blank">
                      <span class="fa-brands fa-instagram"></span>
                      
                      
                    </a>
                  </li>
                  <li class="social-links">
                    <a href="https://www.linkedin.com/company/aging" target="_blank">
                      <span class="fa-brands fa-linkedin"></span>
                      
                      
                    </a>
                  </li>
                  <li class="social-links">
                    <a href="https://www.youtube.com/@Aging-US" target="_blank">
                      <span class="fa-brands fa-youtube"></span>
                      
                      
                    </a>
                  </li>
                  <li class="social-links">
                    <a href="https://www.reddit.com/user/AgingUS/" target="_blank">
                      <span class="fa-brands fa-reddit"></span>
                      
                      
                    </a>
                  </li>
            </ul>
          </div>
        </div>

        <span class="article-submit-button"><a href="https://aging.msubmit.net/" target="_blank" class="submit-article"><span class="icon icon-file"></span>&nbsp;Submit&nbsp;an&nbsp;Article</a></span>

        <div class="mobile-search-and-menu">
          <span class="sidebar-toggle-mobile icon icon-menu" style="margin-right:8px;"></span>
          <a href="/search" style="color:white;"><span id="mobile-search" class="icon icon-search hide-on-large-only"></span></a>
        </div>

      </div>
      <div class="sub-nav sidebar-open secondary">
          <div class="issn">
            <span class="meta">Online ISSN: 1945-4589</span>
          </div>
          <div class="search-box-container" style="float: left; padding:0px;">
            <div class="search-box">
              <span class="searchbox-button icon icon-search"></span>
              <input class="searchbox-field" type="search" placeholder="Search the Journal">
            </div>
          </div>
          <div class="breadcrumbs">
            <span class="sidebar-toggle icon icon-menu"></span>
            <ul>
              <li><a href="/">Home</a></li>
              <li>
                <a href="/archive"> Archive </a>
              </li>
              <li>
                <a href="/issue/v17i8"> Volume 17, Issue 8 </a>
              </li>
              <li>
                <a href="/article/206311"> A protective role of SRC-1 against … </a>
              </li>
            </ul>
          </div>
      </div>
    </header>

    <!-- ========= Main Container ========= -->
    <div class="main-background">
    <div class="main">
      <!-- Sidebar -->
      <div class="sidebar secondary">
        <div id="leftNav-desktop-container">
          <div class="leftNav">
            <ul class="nav-links">
              <li class="nav-title"><span class="title-content">Navigate<span class="sidebar-toggle icon icon-close"></span></span></li>
              <li id="Home"  ><a href="/" class="">Home</a></li>
              <li id="EdBoard"  ><a href="/editorial-board" class="">Editorial Board</a></li>
              <li id="ForAuthors"  ><a href="/editorial-policies" class="">Editorial Policies</a></li>
              <li id="Advance"  ><a href="/advance-publications" class="">Advance Publications</a></li>
              <li id="Current"  ><a href="/current" class="">Current Issue</a></li>
              <li id="Archive"  ><a href="/archive" class="">Archive</a></li>
              <li id="SpecialCollectionsArchive"  ><a href="/special-collections" class="">Special Collections</a></li>
              <li id="Scientific Integrity"  style="color:#a01a57" ><a href="/scientific-integrity" class=" magenta-nav" >Scientific Integrity</a></li>
              <li id="Ethics"  ><a href="/publication-ethics" class="">Publication Ethics Statements</a></li>
              <li id="Interviews"  ><a href="/interviews" class="">Interviews with Outstanding Authors</a></li>
              <li id="PressReleases"  ><a href="/news-room/press-releases" class="">Newsroom</a></li>
              <li id="Conferences"  ><a href="/news-room/conferences" class="">Sponsored Conferences</a></li>
              <li id="Contact"  ><a href="/contact" class="">Contact</a></li>
            </ul>
          </div>
        </div>
        <div id="journal-info-desktop-container">
          <div id="journal-info-box">
            <div class="sidebar-module-box" style="text-align:center; margin-top: 10px;">
              <div style="margin-bottom:6px;">Rapamycin Press LLC is the publisher of Aging: <a href="https://www.impactjournals.com/">www.impactjournals.com</a>.</div>
              <div style="margin-bottom:6px;">Impact Journals is a member of the <a href="https://wellcome.org/" target="_blank">Wellcome Trust List of Compliant Publishers</a>.</div>
              <div>Impact Journals is a member of the <a href="https://www.sspnet.org/">Society for Scholarly Publishing</a>.</div>
            </div>
          </div>
        </div>
        <div id="post-pub-promotion-desktop-container">
          <div id="post-pub-promotion" style="margin-top:15px; background-color: #a11b57; padding: 7px; color: white; font-size: 0.8em; text-align: center;">
            <a href="/for-authors#freepostpublicationpromotionservices" style="color: white;">
              <b>Learn about our FREE Post-Publication Promotion Services</b>
            </a>
          </div>
        </div>
        <div id="banners-left-desktop-container">
          <div id="banners-left">
          </div>
        </div>
      </div>

      <!-- Content -->
      <div id="center-content-column" class="content">
        <div class="content-yield">
          <div id="leftNav-mobile-container" style="text-align:center;"></div>
          <div id="top-social-links-mobile-container" style="text-align:center;"></div>
          <div class="row">
  <div class="page-content col s12 m12 l8">

        <h5 class="article-header">
            <span class="article-type">Research Paper</span>
            <span><a href="/issue/v17i8" style="cursor: pointer;">Volume 17, Issue 8</a></span>
          
          <span>pp 2167&mdash;2188
          
          </span>
        </h5>
      <div id="article-title-and-badge-container" class="valign-wrapper">
            <div id="article-10.18632/aging.206311" data-badge-type="donut" data-template="" data-link-target="_blank" data-badge-popover="left" data-hide-less-than="10" data-altmetric-id="181513134" data-hide-no-mentions="true" data-condensed="true" class="article-altmetric altmetric-embed valign"></div>
        
        <div id="article-container">
          <h1 id="article-title" class="valign">
            A protective role of SRC-1 against aging associated cognitive decline
          </h1>
        </div>
      </div>
      <h4 class="authors">
          <span class="author">
            Hesong Liu<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        <span><a href="#" class="tooltip--triangle" data-tooltip="Equal contribution">*</a><span class="comma">,</span></span>
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Yongjie Yang<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        <span><a href="#" class="tooltip--triangle" data-tooltip="Equal contribution">*</a><span class="comma">,</span></span>
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Jonathan C. Bean<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        <span><a href="#" class="tooltip--triangle" data-tooltip="Equal contribution">*</a><span class="comma">,</span></span>
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Yang He<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Hailan Liu<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Rambabu Majji<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="Jan and Dan Duncan Neurological Research Institute, Texas Children&#x27;s Hospital, Houston, TX 77030, USA">2</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Chen Liang<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Nan Zhang<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Meng Yu<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Longlong Tu<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Qingzhuo Liu<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Yue Deng<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Kristine M. Conde<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Na Yin<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Mengjie Wang<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Yongxiang Li<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Junying Han<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Sanika Vattakuzhiyil Jossy<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Megan Elyse Burt<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Hari Krishna Yalamanchili<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span><span><a href="#" class="tooltip--triangle" data-tooltip="Jan and Dan Duncan Neurological Research Institute, Texas Children&#x27;s Hospital, Houston, TX 77030, USA">2</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
          <span class="author">
            Chunmei Wang<sup class="author-affiliations"><span><a href="#" class="tooltip--triangle" data-tooltip="USDA/ARS Children&#x27;s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA">1</a><span class="comma">,</span></span>
        
        </sup>
      <span class="author-comma">,</span>
          </span>
      </h4>
      
      <span>
        <ul>
              <li>
                <sup>1</sup> USDA/ARS Children's Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, One Baylor Plaza, Houston, TX 77030, USA
              </li>
              <li>
                <sup>2</sup> Jan and Dan Duncan Neurological Research Institute, Texas Children's Hospital, Houston, TX 77030, USA
              </li>
        </ul>
      </span>
      
      
            <div>* Equal contribution</div>
       <h4 class="dates">
            <span class="date"> Received: October 14, 2024 <span class="comma">&emsp;&emsp;&emsp;&emsp;&emsp;</span> </span>
      
              <span class="date"> Accepted: July 21, 2025 <span class="comma">&emsp;&emsp;&emsp;&emsp;&emsp;</span> </span>
      
            <span class="date">Published: August 27, 2025 <span class="comma">&emsp;&emsp;&emsp;&emsp;&emsp;</span> </span>
      
      
      </h4>
      
      
      <span><a href="https://doi.org/10.18632/aging.206311">https://doi.org/10.18632/aging.206311</a></span><br>
      
        <a class="view-citation" data-pii="206311">How to Cite</a>
        <div id="citation-modal-206311" class="modal">
          <div class="modal-content">
            <h2>Cite this Article</h2>
        
            <h3>How to cite</h3>
        
            <p>
              <span class="authors">
                <span class="author">Liu H<span class="author-comma">, </span></span><span class="author">Yang Y<span class="author-comma">, </span></span><span class="author">Bean JC<span class="author-comma">, </span></span><span class="author">He Y<span class="author-comma">, </span></span><span class="author">Liu H<span class="author-comma">, </span></span><span class="author">Majji R<span class="author-comma">, </span></span><span class="author">Liang C<span class="author-comma">, </span></span><span class="author">Zhang N<span class="author-comma">, </span></span><span class="author">Yu M<span class="author-comma">, </span></span><span class="author">Tu L<span class="author-comma">, </span></span><span class="author">Liu Q<span class="author-comma">, </span></span><span class="author">Deng Y<span class="author-comma">, </span></span><span class="author">Conde KM<span class="author-comma">, </span></span><span class="author">et al<span class="author-comma">, </span></span>. A protective role of SRC-1 against aging associated cognitive decline. Aging (Albany NY). 2025 Aug 27; 
         17:2167-2188
        .
         https://doi.org/10.18632/aging.206311 
              </span>
            </p>
        
            <h3>Copy or Download citation:</h3>
        
            <p>Select the format you require from the list below</p>
        
            <ul class="citation-type-list">
              <li><a class="btn-flat citation-button blue darken-4 white-text" data-pii="206311" data-citation-type="plain-text.ris">Plain text</a></li>
              <li><a class="btn-flat citation-button" data-pii="206311" data-citation-type="endnote.enw">EndNote</a></li>
              <li><a class="btn-flat citation-button" data-pii="206311" data-citation-type="ref-manager.ris">Reference Manager</a></li>
              <li><a class="btn-flat citation-button" data-pii="206311" data-citation-type="procite.ris">ProCite</a></li>
              <li><a class="btn-flat citation-button" data-pii="206311" data-citation-type="bibtex.bib">BibteX</a></li>
            </ul>
        
            <textarea id="citation-box-206311" class="citation-box"></textarea>
        
            <p>
              <a class="copy-citation" data-pii="206311" style="cursor:pointer;">Click to copy</a> this citation from the text box above or download the citation:
              <a class="citation-download" href="/article/206311/citation/aging-206311-plain-text.ris" target="_blank">Citation</a> |
              <a class="abstract-citation-download" href="/article/206311/citation/aging-206311-abstract-plain-text.ris" target="_blank">Citation &amp; Abstract</a>
            </p>
          </div>
        </div>
      
      
      
      
      
      
      <p class="justify">
      <span>Copyright: &#xa9; 2025 Liu et al.</span>  This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a> (CC BY 4.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.  
      </p>
      
        <div class="abstract article-text">
          <h3 class="bordered article-header-1" id="abstract">Abstract</h3>
          <p>Introduction: Research indicates a strong correlation between obesity and the risk of dementia, both are linked to steroid receptor coactivator-1 (SRC-1), a transcriptional coactivator.</p>
      <p>Methods: We used RNA sequencing analysis (RNA-Seq) to investigate the transcriptome of SRC-1-KO mice, and identified S100 calcium-binding protein A6 (S100A6), an AD associated gene, as one target of SRC-1. We tested cognitive behaviors in SRC-1-KO mice and mice with a humanized SRC-1 mutation (SRC-1<sup>L1376P</sup>), and performed promoter luciferase assays on S100A6.</p>
      <p>Results: Loss of SRC-1 caused alterations in gene signatures that are commonly associated with neurodegenerative diseases, including AD, and diminished the neural plasticity of the hippocampal CA1 neurons. Both SRC-1-KO and SRC-1<sup>L1376P</sup> mice displayed early signs of contextual memory impairment at 6 months of age. Mechanistically, SRC-1 significantly promoted the expression S100A6.</p>
      <p>Conclusion: We identified a protective role of SRC1 against aging associated cognitive decline, potentially by promoting the expression of S100A6.</p>
          <div id="fulltext"></div>
        </div>


          

          <div class="article-text  ">

                <div class="section-container" id="introduction">
                  <h2 class="article-header-1"> Introduction</h2>
                  
                        <p>About 36% of the elderly population (at an age of 70–75) experience mild memory loss, and about 32% are affected by Alzheimer’s disease (AD) [<a href="#r1"  class="anchor">1</a>]. Owing to the aging of populations worldwide, dementia is reaching epidemic proportions, with a large human, social and economic burden. AD is the most common cause of severe memory loss in the elderly. Despite the tremendous efforts in the field of cognitive regulation, the pathophysiology underlying memory decline is not fully understood, and effective treatments are limited.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>Research indicates a strong correlation between obesity and an increased risk of dementia and Alzheimer’s disease [<a href="#r2"  class="anchor">2</a>–<a href="#r6"  class="anchor">6</a>]. The brain is rich in various nuclear receptors and transcription factors that play a crucial role in maintaining metabolic equilibrium and cognitive functions. Notably, thyroid hormone receptor [<a href="#r7"  class="anchor">7</a>], glucocorticoid receptor [<a href="#r8"  class="anchor">8</a>], estrogen receptor [<a href="#r9"  class="anchor">9</a>], peroxisome proliferator-activated receptor gamma [<a href="#r10"  class="anchor">10</a>], signal transducer and activator of transcription-3 [<a href="#r11"  class="anchor">11</a>], and Forkhead Box O1 [<a href="#r12"  class="anchor">12</a>], etc. are among those implicated in both metabolic and Alzheimer’s diseases progression. These nuclear receptors and transcription factors depend on coactivators like steroid receptor coactivator-1, 2 and 3 (SRC-1, 2 and 3) [<a href="#r13"  class="anchor">13</a>] for their transcriptional activities. SRC proteins are prevalent in the brain, especially in areas like the hippocampus and hypothalamus [<a href="#r14"  class="anchor">14</a>, <a href="#r15"  class="anchor">15</a>]. Studies have shown that the deletion of SRC-1 (encoded by <i>Ncoa1</i>) gene globally or from the hypothalamus in mice leads to obesity [<a href="#r16"  class="anchor">16</a>, <a href="#r17"  class="anchor">17</a>]. A particular gene variant, SRC-1<sup>L1376P</sup> mutation, has been linked to early-onset obesity in children (body mass index standard deviation score (BMI SDS) >3.5; age of onset <10 years), and has been confirmed to induce obesity in mice as well [<a href="#r17"  class="anchor">17</a>, <a href="#r18"  class="anchor">18</a>]. Despite that loss of SRC-1 does not accelerate the development of AD [<a href="#r19"  class="anchor">19</a>], another study showed that RNAi-mediated knockdown of SRC-1 in the hippocampal CA1 impairs memory [<a href="#r20"  class="anchor">20</a>, <a href="#r21"  class="anchor">21</a>]. Thus, the dysfunction of SRC-1 is linked to both obesity and memory loss. However, it is unclear whether and how SRC-1 contributes to the aging associated dementia.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>Earlier studies have indicated a reduction in SRC-1 expression in the brains of middle-aged individuals [<a href="#r22"  class="anchor">22</a>]. In this study, we confirmed the age associated decline of SRC-1 and further explored the differences in brain gene expression profiles between wild type C57BL/6 (WT) and SRC-1-KO mice. We performed cognitive behavioral tests on WT mice, SRC-1-KO mice and mice with the humanized SRC-1<sup>L1376P</sup> mutation across various age stages to assess the role of SRC-1 in cognitive functions. Finally, we identified the stimulatory effect of SRC-1 on the expression of S100A6 and S100A11, which are genes potentially associated with AD and memory deficits.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                </div>
                <div class="section-container" id="methods">
                  <h2 class="article-header-1"> Methods</h2>
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Mice</h3>
                          
                                <p>Care of all animals and procedures were approved by the Institutional Animal Care and Use Committee of Baylor College of Medicine (AN-5479 and AN-6098). Mice, including SRC-1-KO mice [<a href="#r16"  class="anchor">16</a>] and SRC-1<sup>L1376P</sup> mutant mice [<a href="#r17"  class="anchor">17</a>], were housed in a temperature-controlled room at 22–24°C using a 12-h light, 12-h dark cycle. All these mice were fed with regular chow (5V5R, PicoLab). Food and water were provided ad-libitum.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> RNA-Seq and analysis</h3>
                          
                                <p>Total hypothalamic RNA was isolated from SRC-1-KO mice and littermates at the age of 20 weeks using the RNeasy Lipid Tissue Mini Kit (Qiagen). Three RNA samples for each group were sent to Genomic and RNA Profiling Core (GARP) at Baylor College of Medicine for sequencing. One hundred-fifty base pair paired end reads were aligned to Genome Reference Consortium Mouse Build 39 reference genome (Mus musculus genome assembly GRCm39 NCBI. <a href="https://www.ncbi.nlm.nih.gov/data-hub/assembly/GCF_000001635.27/" target="_blank">https://www.ncbi.nlm.nih.gov/data-hub/assembly/GCF_000001635.27/</a>) using STAR 2.7.9a using option “--outSAMtype BAM SortedByCoordinate” [<a href="#r23"  class="anchor">23</a>]. Output from STAR was then quantified using “featureCounts” function from Subread v2.0.3 using default options [<a href="#r24"  class="anchor">24</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Differential expression was calculated using DEseq2 1.36.0 with test set to “Wald” [<a href="#r25"  class="anchor">25</a>] with R 4.2.2 within RStudio 2022.07.2 Build 576. Differential expression results were filtered for genes that were 1.5-fold either up or down with an adjusted <i>p</i>-value <0.05 using dplyr 1.0.10. These genes were used to produce two list of genes that were then used to create molecular function gene ontology utilizing “gost” function from gprofiler2 0.2.1 with options “organism = ‘mmusculus’, ordered_query = FALSE, evcodes = TRUE, correction_method = ‘gSCS’, domain_scope = ‘annotated’, exclude_iea = TRUE, sources = ‘GO:MF’”.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>An additional enrichment analysis was conducted using Gene Set Enrichment Analysis (GSEA, version 4.3.2) [<a href="#r26"  class="anchor">26</a>] with the “preranked” option. Genes were first rank ordered using the algorithm log<sub>2</sub>(fold change) X - log<sub>10</sub> (adjusted <i>p</i>-value). Options on GSEA were set to: gene set database “c2.cp.kegg.v2022.1.Hs.symbols.gmt”, chip platform Mouse_Gene_Symbol_Remapping_Human_Orthologs_MsigDB.v2022.1.Hs.chip”, enrichment statistic “classic” and number of permutations “100,000”. Other options were left on default settings. RNA-Seq results were visualized using ggplot2.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Alternative splicing was detected using rMATS turbo (v4.1.24) [<a href="#r27"  class="anchor">27</a>], in Python 2.7.18. Genes splicing events were considered significant if the absolute inclusion level difference was greater than 0.2 and adjusted <i>p</i>-value was less than 0.05.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Secondary analysis of published scRNA-Seq data</h3>
                          
                                <p>Data were obtained from GSE152506 [<a href="#r28"  class="anchor">28</a>]. Raw counts were first normalized using the log<sub>2</sub> counts per 10,000 formula. Data from either hypothalamus or hippocampus were subset then compared across age using FindMarkers feature of Seurat 5.1.0, Wilcox test followed by Bonferroni correction for multiple comparisons [<a href="#r29"  class="anchor">29</a>]. Data were plotted using VlnPlot feature of Seurat with boxplots superimposed on top.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Secondary analysis of published bulk RNA-Seq data</h3>
                          
                                <p>Gene expression data for both human and mouse brain tissues were obtained from publicly available sources. Human bulk tissue RNA-seq data were downloaded from the GTEx portal (<a href="https://www.gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression" target="_blank">https://www.gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression</a>), including brain regions such as the Cerebellum, Cortex, Frontal Cortex, Hippocampus, and Hypothalamus. Corresponding sample metadata, including donor age information, were also downloaded from GTEx and used to stratify samples into two age groups: 20–40 years (combining 20–29 and 30–39 years) and 60–80 years (combining 60–69 and 70–79 years). Differential gene expression (DEG) analysis was performed using DESeq2 [<a href="#r25"  class="anchor">25</a>] on normalized count data. Log-transformed expression values were visualized with violin plots, with age groups on the x-axis and expression on the y-axis. Statistical significance was denoted by asterisks (<sup>*</sup> for <i>p</i> < 0.05, <sup>**</sup> for <i>p</i> < 0.01). For mouse data, hippocampal gene expression profiles were obtained from the GEO dataset GSE179698, which includes RNA-seq samples from 6- and 18-month-old mice. Expression values for Ncoa1, S100a6, and S100a11 were extracted from the dataset, and log-transformed values were visualized using box-and-whisker plots.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Electrophysiology</h3>
                          
                                <p>Mice (males, 6 weeks of age) were perfused and brain slices were prepared for electrophysiology recording as we did before. Briefly, pyramidal neurons in CA1, identified based on their location and morphology, were visualized and recorded. Whole-cell recordings were performed on CA1 pyramidal neurons. Evoked EPSCs (eEPSCs) and EPSC based LTP were recorded as we did before [<a href="#r30"  class="anchor">30</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Immunofluorescence</h3>
                          
                                <p>WT mice (males, 10 weeks of age) were perfused and brains were sectioned and processed for immunohistochemistry staining for SRC-1 and three cell markers. Briefly, brain sections were blocked (3% Normal donkey serum) for 1 h, incubated with Rabbit anti-SRC-1 (#2191, Cell Signaling Technology) on shaker at 4°C for overnight, followed by 3 washes with 1X PBS and then the incubation of the donkey anti-rabbit AlexaFluor 488 (A21206, Invitrogen) for 2 h. After three washes, each series of brain sections were then incubated with one of the cell marker antibodies respectively. Cell marker antibodies include Mouse anti-NeuN (ab104224, Abcam) as neuron maker, Chicken anti-GFAP (ab4674, Abcam) as astrocyte maker and Goat anti-Iba1 (ab5076, Abcam) as microglial cell marker. All the brain sections were then incubated with the donkey anti-mouse/chicken/goat AlexaFluor 594 (A21203/A11042/A11058, Invitrogen) for 2 h. After the last wash and dry, slides were cover-slipped, and images of the hippocampus were captured using a fluorescence microscope.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Behavioral tests</h3>
                          
                                <p>The Novel Object Recognition (NOR) test was conducted over two consecutive days to assess memory in mice using a modified protocol [<a href="#r31"  class="anchor">31</a>]. On the first day, each mouse was acclimated to the testing room for 30 minutes, followed by a 20-minute habituation period in an empty Open Field (OF) box (40.64 cm × 40.64 cm). Subsequently, two identical objects were placed in opposite corners of the OF box. The mouse was then allowed to explore the arena and the objects freely for 15 minutes with an overhead camera to record its behavior for subsequent analysis. After the 15 minutes session, the mouse was returned to its home cage, and the OF box and objects were thoroughly cleaned with soap and water to eliminate olfactory cues. On the second day, following a 30-minute acclimatization to the testing room, the mouse was then reintroduced to the same OF box. One of the familiar objects was replaced with a novel object that differed in both shape and color. The mouse was again allowed to explore the arena for 15 minutes, during which behavior was recorded. After the session, the mouse was returned to its home cage, and the testing apparatus was cleaned as described above. The Discrimination Index was calculated as the time interacted with the novel object divided by the total time that the mouse interacted with both objects. The water maze test (RAWM) and fear conditioning test were performed as we did previously [<a href="#r30"  class="anchor">30</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Q-PCR validation of gene expression in the hypothalamus and hippocampus</h3>
                          
                                <p>To examine gene expression, C57BL/6 control and SRC-1-KO mice (male, 4 months of age) were sacrificed, and the hypothalamus and hippocampus were quickly collected. Total RNA was isolated using TRIzol Reagent (Invitrogen) and 2 μg of total RNA was reverse-transcribed to cDNA using a High-Capacity cDNA Reverse Transcription Kits (Invitrogen). Q-PCR was performed on a CFX384 Real-Time System (Bio-Rad) using SsoADV SYBR Green Supermix (Bio-Rad). Primer sequences for S100a6: forward 5′-GAAGGTGACAAGCACACCCT-3′ and reverse: 5′-CCCAGGAAGGCGACATACTC-3′; for S100a11: forward 5′-AAGTACAGCGGGAAGGATGGA-3′ and reverse 5′-ATGCGGTCAAGGACACCAG-3′; for Cyclophilin: forward 5′-TGGAGAGCACCAAGACAGACA-3′ and reverse 5′-TGCCGGAGTCGACAATGAT-3′. The expressions of S100a6 and S100a11 were normalized to the house-keeping gene Cyclophilin.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Western blot</h3>
                          
                                <p>Hypothalamus and hippocampus from young (4 months of age) and aged (13 months of age) were collected and lysed with lysis buffer: 50 mM Tris, 50 mM KCL, 10 mM EDTA, 1% NP-40, supplied with protease inhibitor cocktail (Roche). Total protein tissue lysate (20 ug) from each mouse was loaded for SDS-PAGE and then detected with SRC-1 (128E7) Rabbit mAb (#2191, Cell signaling, at 1:2000) and HRP-linked anti-rabbit IgG Antibody (at 1:10000, Cell signaling).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Luciferase assay</h3>
                          
                                <p>Neuro 2A (mouse neuroblastoma cell line) and immortalized SRC-1-KO MEF cells [<a href="#r17"  class="anchor">17</a>] were cultured in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum (Atlanta), 100 IU/ml penicillin and 100 ng/ml streptomycin. Luciferase reporter plasmid for S100a6 (1.66K, −1744 to −81, with primer pairs: forward 5′-AAAGGCCGTGAGAGCTAGGA-3′ and reverse 5′-TGAGGCAGTCAGTCTCAAGC-3′) and S100a11 (1.2K, −1292 to −76, with primer pairs: forward 5′-AGCTGAAATTCCAAGGGCCA-3′ and reverse 5′-TCCCCATGTCGGTGCTCTA-3′), were cloned as previous described [<a href="#r17"  class="anchor">17</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Luciferase assay using Neuro 2A cells was performed to test if the promoters of S100a6 and S100a11 can be regulated by these transcription factors, including AP2 (gift from Robert Tjian, Addgene plasmid # 12100), Sp1 (gift from Guntram Suske, Addgene plasmid # 24543), TCF1 (gift from Kai Ge, Addgene plasmid # 40620), KLF4 (gift from Derrick Rossi, Addgene plasmid # 26815) and PU.1 (Gift from Qiang Tong) [<a href="#r32"  class="anchor">32</a>]. Neuro 2A cells were transfected with 800 ng of the luciferase reporter plasmid combined with 200 ng of indicated transcription factor plasmid or the control empty plasmid using the Lipofectamine LTX (Invitrogen). Cells were lysed 40 hours post-transfection, and the luciferase activity was measured using the Luciferase<sup>®</sup> Reporter Assay System (Promega).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>To test if SRC-1 enhances the expression of S100a6 and S100a11, SRC-1-KO MEF cells were transfected with 700 ng of the luciferase reporter plasmid combined with 200 ng of indicated transcription factor plasmid and 100 ng of pCR3.1-SRC-1 or the control empty plasmid.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                </div>
                <div class="section-container" id="results">
                  <h2 class="article-header-1"> Results</h2>
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> SRC-1 is associated with neurodegenerative diseases</h3>
                          
                                <p>SRC-1 has been implicated to regulate both metabolic balance and cognitive functions. Importantly, the expression of SRC-1 is significantly declined with aging [<a href="#r22"  class="anchor">22</a>]. To explore the potential contribution of SRC-1 to aging associated diseases, including metabolic dysregulations and dementia, we performed RNA-Seq analyses using samples obtained from the hypothalamus of SRC-1-KO mice and C57BL/6 control mice (<a href="#f1"  class="anchor">Figure 1A</a>), and the data has been uploaded to GEO (GSE278158). As a validation, SRC-1 (coded by Ncoa1) was significantly depleted in the KO mice (Supplementary <a href="#f1"  class="anchor">Figure 1A</a>). Consistent with the protective role of SRC-1 in metabolic regulation, Pomc, Lepr and downstream Stat3 were significantly decreased in SRC-1-KO mice (Supplementary <a href="#f1"  class="anchor">Figure 1B</a>–<a href="#f1"  class="anchor">1D</a>). This is consistent with our previous study showing decreased leptin sensitivity in mice with deletion of SRC-1 selectively from POMC neurons, a population of hypothalamic neurons essential for metabolic regulation [<a href="#r17"  class="anchor">17</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f1">
                                      <a href="/figure/206311/f1"><img alt="Differential gene expression in the hypothalamus of SRC-1-KO mice. (A) Diagram depicting mouse brain in sagittal view (top) and coronal view (bottom) with red dotted lines indicating hypothalamic region collected for RNA-seq experiment. (B) Volcano plot of RNA-seq results (center) with log2 fold change plotted on the x-axis and -log10 corrected p-value (Padj) plotted on the y-axis. Horizontal dashed line indicates Padj &#x3D; 0.05. Two vertical lines indicate 1.5-fold either up or down. Each dot represents one gene. Genes plotted in red represent genes that are both 1.5-fold different with a Padj Padj &#x3D; 1e-28 and indicates genes plotted above this line are out of the scale of the panel. Large orange arrows mark the number of genes that are 1.5-fold lower (left) or higher (right) with a Padj C) Gene ontology for molecular function using genes 1.5-fold down with Padj Padj D, E) Violin plots for S100a6 (D) and S100a11 (D) normalized to transcripts per million (TPM). * and **, Padj F) Gene set enrichment analysis using Kyoto encyclopedia of genes and genomes (KEGG) as output with a rank ordered gene list, using log2 fold change X -log10 Padj as the algorithm to rank genes, as input." class="full-text-image" src="/article/206311/figure/f1/large"/></a>
                                      <p class="small-font"><span><b>Figure 1.</b></span>  <b>Differential gene expression in the hypothalamus of SRC-1-KO mice.</b> (<b>A</b>) Diagram depicting mouse brain in sagittal view (top) and coronal view (bottom) with red dotted lines indicating hypothalamic region collected for RNA-seq experiment. (<b>B</b>) Volcano plot of RNA-seq results (center) with log<sub>2</sub> fold change plotted on the x-axis and -log<sub>10</sub> corrected <i>p</i>-value (<i>P</i>adj) plotted on the y-axis. Horizontal dashed line indicates <i>P</i>adj = 0.05. Two vertical lines indicate 1.5-fold either up or down. Each dot represents one gene. Genes plotted in red represent genes that are both 1.5-fold different with a <i>P</i>adj < 0.05. Light grey dashed horizontal line marks <i>P</i>adj = 1e-28 and indicates genes plotted above this line are out of the scale of the panel. Large orange arrows mark the number of genes that are 1.5-fold lower (left) or higher (right) with a <i>P</i>adj < 0.05. (<b>C</b>) Gene ontology for molecular function using genes 1.5-fold down with <i>P</i>adj < 0.05 (top) or 1.5-fold up with <i>P</i>adj < 0.05 (bottom). (<b>D</b>, <b>E</b>) Violin plots for S100a6 (<b>D</b>) and S100a11 (<b>D</b>) normalized to transcripts per million (TPM). <sup>*</sup> and <sup>**</sup>, <i>P</i>adj < 0.05 or 0.01. (<b>F</b>) Gene set enrichment analysis using Kyoto encyclopedia of genes and genomes (KEGG) as output with a rank ordered gene list, using log2 fold change X -log10 <i>P</i>adj as the algorithm to rank genes, as input.<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                                <p>Applying a threshold of 1.5-fold difference and a corrected <i>p</i>-value <0.05, there were 396 genes down regulated and 103 genes up regulated in SRC-1-KO mice (<a href="#f1"  class="anchor">Figure 1B</a> and Supplementary data 1). We used these genes to perform Gene Set enrichment Analysis (GSEA) and Gene ontology (GO) analysis. The enrichment analysis for molecular functions revealed dysfunction in extracellular matrix binding, as well as intracellular cytoskeletal binding gene profiles. Interestingly, S100 protein binding was the most significant downregulated term, and the structure constituent of myelin sheath was the most significant upregulated term (<a href="#f1"  class="anchor">Figure 1C</a>). Importantly, S100 proteins and myelin functions are both implicated in the pathology of AD [<a href="#r33"  class="anchor">33</a>–<a href="#r37"  class="anchor">37</a>]. In particular, SRC-1-KO mice express significantly lower S100A6 and S100A11 (<a href="#f1"  class="anchor">Figure 1D</a>, <a href="#f1"  class="anchor">1E</a>), and both genes have been demonstrated neuroprotective effects in neurodegenerative diseases [<a href="#r35"  class="anchor">35</a>, <a href="#r38"  class="anchor">38</a>–<a href="#r47"  class="anchor">47</a>].</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Further, Kyoto encyclopedia of genes and genomes (KEGG) analysis revealed that gene sets related to neurodegenerative disorders, including Alzheimer’s, were enriched, while the gene set for spliceosome was downregulated in SRC-1-KO mice (<a href="#f1"  class="anchor">Figure 1F</a>). Utilizing multivariate analysis of transcript splicing with an absolute inclusion level difference >0.2 and FDR <0.05, we found there were 343 significant splicing events (Supplementary <a href="#f1"  class="anchor">Figure 1E</a>) corroborating an altered spliceosome activity. Importantly, recent studies suggest that dysregulation of spliceosome also increases the risk of AD [<a href="#r48"  class="anchor">48</a>–<a href="#r50"  class="anchor">50</a>]. Although these splicing events are not directly involved in AD, GO analysis showed that they are involved in neuron projection organization (Supplementary <a href="#f1"  class="anchor">Figure 1E</a>), which is essential for normal neuron functions.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Together, the changes of gene profiles in SRC-1-KO mice suggest higher risks of neurodegenerative diseases including AD.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> SRC-1 regulates neural plasticity of the hippocampal CA1 neurons</h3>
                          
                                <p>It has been reported that siRNA-mediated knockdown of SRC-1 in the hippocampal CA1 impairs memory in mice, associated with decreased CA1 synapse density, postsynaptic density thickness, and long-term potentiation (LTP) [<a href="#r20"  class="anchor">20</a>]. This finding led us to focus on the hippocampal CA1 region. Here we found that both neurons (marked by NeuN) and astrocytes (marked by GFAP) in the hippocampal CA1 abundantly expressed SRC-1, while microglial cells (marked by Iba) did not (<a href="#f2"  class="anchor">Figure 2</a>).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f2">
                                      <a href="/figure/206311/f2"><img alt="The expression of SRC-1 in the hippocampus. (A–C) Co-staining of SRC-1 (green, A) and a neuron marker (NeuN, red, B), and merge (C) in the hippocampal CA1 region. (D) is high magnification indicating co-expression of SRC-1 in NeuN labeled neurons. (E–G) Co-staining of SRC-1 (green, E) and an astrocyte marker (GFAP, red, F), and merge (G) in the hippocampal CA1 region. (H) is high magnification indicating co-expression of SRC-1 in GFAP labeled astrocytes. (I–K) Co-staining of SRC-1 (green, I) and a microglial marker (Iba, red, J), and merge (K) in the hippocampal CA1 region. (L) is high magnification indicating no double-labelling." class="full-text-image" src="/article/206311/figure/f2/large"/></a>
                                      <p class="small-font"><span><b>Figure 2.</b></span>  <b>The expression of SRC-1 in the hippocampus.</b> (<b>A</b>–<b>C</b>) Co-staining of SRC-1 (green, <b>A</b>) and a neuron marker (NeuN, red, <b>B</b>), and merge (<b>C</b>) in the hippocampal CA1 region. (<b>D</b>) is high magnification indicating co-expression of SRC-1 in NeuN labeled neurons. (<b>E</b>–<b>G</b>) Co-staining of SRC-1 (green, <b>E</b>) and an astrocyte marker (GFAP, red, <b>F</b>), and merge (<b>G</b>) in the hippocampal CA1 region. (<b>H</b>) is high magnification indicating co-expression of SRC-1 in GFAP labeled astrocytes. (<b>I</b>–<b>K</b>) Co-staining of SRC-1 (green, <b>I</b>) and a microglial marker (Iba, red, <b>J</b>), and merge (<b>K</b>) in the hippocampal CA1 region. (<b>L</b>) is high magnification indicating no double-labelling.<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                                <p>We further explored neural plasticity in the hippocampal CA1 neurons. To this end, we used a high-frequency field stimulation (HFS) protocol to induce long-term potentiation (LTP) of evoked excitatory post-synaptic current (EPSC) in CA1 neurons from WT mice, as indicated by a sustained increase of EPSC after the HFS stimulation. However, in SRC-1-KO mice, LTP was significantly blunted (<a href="#f3"  class="anchor">Figure 3</a>). These results predict a protective role of hippocampal SRC-1 in cognitive functions.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f3">
                                      <a href="/figure/206311/f3"><img alt="Impaired LTP in hippocampal CA1 neurons of SRC-1-KO mice. (A) Typical EPSC traces before (lighter curve) and after (darker curve) LTP induction in CA1 neurons from WT or SRC-1-KO mice. (B) Magnitude of EPSC elevations before (0–5 min) and after LTP induction (5–50 min). (C) Averaged EPSC elevations during 45–50 min in (B). N &#x3D; 5-6 neurons from 3 mice. **P t-tests." class="full-text-image" src="/article/206311/figure/f3/large"/></a>
                                      <p class="small-font"><span><b>Figure 3.</b></span>  <b>Impaired LTP in hippocampal CA1 neurons of SRC-1-KO mice.</b> (<b>A</b>) Typical EPSC traces before (lighter curve) and after (darker curve) LTP induction in CA1 neurons from WT or SRC-1-KO mice. (<b>B</b>) Magnitude of EPSC elevations before (0–5 min) and after LTP induction (5–50 min). (<b>C</b>) Averaged EPSC elevations during 45–50 min in (<b>B</b>). <i>N</i> = 5-6 neurons from 3 mice. <sup>**</sup><i>P</i> < 0.01 in unpaired two-tailed <i>t</i>-tests.<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> SRC-1 prevents aging-associated memory loss</h3>
                          
                                <p>To fully evaluate the function of SRC-1, we compared the cognitive behaviors among mice with either global deletion of SRC-1 (SRC-1-KO) or knock-in of a humanized loss-of-function point mutation (SRC-1<sup>L1376P</sup>) [<a href="#r17"  class="anchor">17</a>]. Similar to human, mice displayed an aging associated cognition decline. The contextual memory of WT mice was significantly decreased after 18 months of age (<a href="#f4"  class="anchor">Figure 4A</a>, <a href="#f4"  class="anchor">4B</a>), as indicated by the decrease of freezing behavior in the contextual fear conditioning test. However, this reduction did not happen in SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice, because their contextual memory had already been significantly impaired as early as 6 months of age compared to WT mice (<a href="#f4"  class="anchor">Figure 4C</a> and Supplementary <a href="#f2"  class="anchor">Figure 2A</a>, <a href="#f2"  class="anchor">2B</a>). Consistently, the survival rate dropped with aging in all mice, and it was significantly lower in SRC-1-KO mice and displayed a non-significant decrease trend in SRC-1<sup>L1376P</sup> mice, compared to WT control mice (<a href="#f4"  class="anchor">Figure 4D</a>). These results together supported an early-aging phenotypes caused by SRC-1 deficiency.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f4">
                                      <a href="/figure/206311/f4"><img alt="Impaired cognitive functions of SRC-1-KO mice and SRC-1L1376P mice at different ages. (A–C) Freezing time in the fear conditioning test in all mice at different ages (A), in WT mice at different ages (B), and in all groups of mice at 6 months of age (C). N &#x3D; 8–12 mice/group. Data are presented as mean ± SEM and/or with individual data points. *P **P ***P A) or one way ANOVA analysis followed by Dunnett&#x27;s multiple comparisons test (B, C). (D) The surviving curve of WT mice, SRC-1-KO mice and SRC-1L1376P mice. *P E, F) Discrimination score is defined as the ratio of time spent with the novel object during test vs. (training + test) in the novel object test in WT, SRC-1L1376P and SRC-1-KO mice at different ages (E) and at 18 months of age (F). N &#x3D; 8–12 mice/group. Data are presented as mean ± SEM with individual data points. **P ****P E) or one way ANOVA analysis followed by Dunnett&#x27;s multiple comparisons test (F)." class="full-text-image" src="/article/206311/figure/f4/large"/></a>
                                      <p class="small-font"><span><b>Figure 4.</b></span>  <b>Impaired cognitive functions of SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice at different ages.</b> (<b>A</b>–<b>C</b>) Freezing time in the fear conditioning test in all mice at different ages (<b>A</b>), in WT mice at different ages (<b>B</b>), and in all groups of mice at 6 months of age (<b>C</b>). <i>N</i> = 8–12 mice/group. Data are presented as mean ± SEM and/or with individual data points. <sup>*</sup><i>P</i> < 0.05; <sup>**</sup><i>P</i> < 0.01; <sup>***</sup><i>P</i> < 0.001 in in two-way ANOVA analyses followed by Tukey’s multiple comparisons test (<b>A</b>) or one way ANOVA analysis followed by Dunnett's multiple comparisons test (<b>B</b>, <b>C</b>). (<b>D</b>) The surviving curve of WT mice, SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice. <sup>*</sup><i>P</i> < 0.05 in Logrank test for trend test. (<b>E</b>, <b>F</b>) Discrimination score is defined as the ratio of time spent with the novel object during test vs. (training + test) in the novel object test in WT, SRC-1<sup>L1376P</sup> and SRC-1-KO mice at different ages (<b>E</b>) and at 18 months of age (<b>F</b>). <i>N</i> = 8–12 mice/group. Data are presented as mean ± SEM with individual data points. <sup>**</sup><i>P</i> < 0.01; <sup>****</sup><i>P</i> < 0.0001 in in two-way ANOVA analyses followed by Tukey’s multiple comparisons test (<b>E</b>) or one way ANOVA analysis followed by Dunnett's multiple comparisons test (F).<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                                <p>In the novel object recognition (NOR) test, SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice displayed significant decreased discrimination score between novel object and old object at 18 months of age, indicating impaired cognition (<a href="#f4"  class="anchor">Figure 4E</a>, <a href="#f4"  class="anchor">4F</a>). However, both SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice displayed similar performance to WT mice in the Radial Arm Water Maze test (RAWM), despite a non-significant trend of impaired spatial memory (Supplementary <a href="#f2"  class="anchor">Figure 2C</a>, <a href="#f2"  class="anchor">2D</a>).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> SRC-1 expression during aging</h3>
                          
                                <p>We demonstrated a protective role of SRC1 against aging associated cognitive decline, and we further explored the molecular mechanisms. We reanalyzed human public data libraries (GTEx portal) and demonstrated an age associated decline of SRC-1 expression in different brain regions, including the hippocampus, hypothalamus and cerebellum, and a non-significant decrease trend in the cortex (<a href="#f5"  class="anchor">Figure 5A</a>–<a href="#f5"  class="anchor">5D</a>). We reanalyzed published spatial RNA-Seq data to compare gene profiles between young (3 months) and old (18 months) mice brains [<a href="#r28"  class="anchor">28</a>]. Interestingly, we found that the expression of SRC-1 was significantly decreased in both the hypothalamus and hippocampus of aged mice compared to young mice (Supplementary <a href="#f3"  class="anchor">Figure 3A</a>, <a href="#f3"  class="anchor">3B</a>). S100A6 displayed a non-significant decrease trend in the hippocampus and a significant decrease in the cerebellum of aged human samples, as well as a significant decrease in the hypothalamus of aged mouse samples (<a href="#f5"  class="anchor">Figure 5E</a>–<a href="#f5"  class="anchor">5H</a> and Supplementary <a href="#f3"  class="anchor">Figure 3D</a>, <a href="#f3"  class="anchor">3E</a>). S100A11 displayed no changes or opposite changes (<a href="#f5"  class="anchor">Figure 5I</a>–<a href="#f5"  class="anchor">5L</a> and Supplementary <a href="#f3"  class="anchor">Figure 3G</a>, <a href="#f3"  class="anchor">3H</a>). Further, secondary analysis of public bulk RNA-Seq data of mice (GSE179698) revealed a decrease trend of both SRC-1 and S100A6 but not S100A11 in the hippocampus of 18-month aged mice compared to 6-month young mice (Supplementary <a href="#f3"  class="anchor">Figure 3C</a>, <a href="#f3"  class="anchor">3F</a>, <a href="#f3"  class="anchor">3I</a>). To further confirm the age-associated decline of SRC-1 in mice, we detected the expression of SRC-1 protein using western blot. As expected, SRC-1 protein was significantly lower in the hippocampus of 13-month aged mice than that of 4-month young mice (<a href="#f5"  class="anchor">Figure 5M</a>, <a href="#f5"  class="anchor">5N</a>).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f5">
                                      <a href="/figure/206311/f5"><img alt="Aging-associated changes in the expression of SRC-1, S100A6 and S100A11 in human. Secondary analysis of published RNA-Seq data revealed the differential gene expression between young (20–40 years of age) and old (60–80 years of age) human. (A–D) The expression of SRC-1 coding gene NCOA1, (E–H) S100A6 coding gene and (I–L) S100A11 coding gene in the hippocampus, hypothalamus, cerebellum and cortex of young vs aged human brains, respectively. * and **P M, N) Western Blot detection (M) and quantification (N) of SRC-1 protein levels compared to housekeeping protein β-actin in the hippocampus of young (3 months of age) and aged mice (13 months of age). N &#x3D; 4–5 mice. *P t-tests." class="full-text-image" src="/article/206311/figure/f5/large"/></a>
                                      <p class="small-font"><span><b>Figure 5.</b></span>  <b>Aging-associated changes in the expression of SRC-1, S100A6 and S100A11 in human.</b> Secondary analysis of published RNA-Seq data revealed the differential gene expression between young (20–40 years of age) and old (60–80 years of age) human. (<b>A</b>–<b>D</b>) The expression of SRC-1 coding gene NCOA1, (<b>E</b>–<b>H</b>) S100A6 coding gene and (<b>I</b>–<b>L</b>) S100A11 coding gene in the hippocampus, hypothalamus, cerebellum and cortex of young vs aged human brains, respectively. <sup>*</sup> and <sup>**</sup><i>P</i> < 0.05 or 0.01 in Wald test from DESeq2 with Benjamini-Hochberg correction for multiple comparisons. (<b>M</b>, <b>N</b>) Western Blot detection (<b>M</b>) and quantification (<b>N</b>) of SRC-1 protein levels compared to housekeeping protein β-actin in the hippocampus of young (3 months of age) and aged mice (13 months of age). <i>N</i> = 4–5 mice. <sup>*</sup><i>P</i> < 0.05 in unpaired two-tailed <i>t</i>-tests.<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                                <p>Meanwhile, similar to reduced synaptic proteins by acute knockdown of SRC-1 in the hippocampus, the expression of Synaptophysin, GluR1 and PSD-95 [<a href="#r20"  class="anchor">20</a>] was also decreased in both the hippocampus and the hypothalamus of aged mice, suggesting aging associated decline of synaptic functions (Supplementary <a href="#f4"  class="anchor">Figure 4A</a>–<a href="#f4"  class="anchor">4H</a>). However, these synaptic proteins were not changed by SRC-1 deletion (Supplementary <a href="#f4"  class="anchor">Figure 4I</a>–<a href="#f4"  class="anchor">4L</a>). Based on the non-association between the expression of SRC-1 and synaptic proteins with chronic loss of SRC-1, the early-aging cognitive decline in SRC-1-KO mice should be contributed by other molecular mechanisms than these synaptic proteins.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>Together, S100A6 and SRC-1 displayed synchronously decrease in both SRC-1-KO mice and aging mice, implying S100A6 as a potential downstream target gene of SRC-1.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> SRC-1 stimulates S100A6 expression</h3>
                          
                                <p>RNA-Seq data identified that S100A6 and S100A11 were downregulated by SRC-1 deficiency in the hypothalamus. We validated the decreased expression of these two gene in the hypothalamus (<a href="#f6"  class="anchor">Figure 6A</a> and Supplementary <a href="#f5"  class="anchor">Figure 5A</a>). The hippocampus is the major center of cognition, and hippocampus CA1 neurons contribute to both NOR [<a href="#r51"  class="anchor">51</a>, <a href="#r52"  class="anchor">52</a>] and contextual fear [<a href="#r53"  class="anchor">53</a>, <a href="#r54"  class="anchor">54</a>]. Based on the high expression of SRC-1 in the cognition center, hippocampus CA1 neurons, we further determined whether SRC-1 regulates S100A6 and S100A11 in the hippocampus, too. S100A6 was significantly decreased (<a href="#f6"  class="anchor">Figure 6B</a>), and S100A11 displayed a decrease trend (Supplementary <a href="#f5"  class="anchor">Figure 5B</a>), in the hippocampus of SRC-1-KO mice. These results suggest S100A6 and S100A11 as two potential downstream targets of SRC-1, a co-factor for transcription factors.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <!-- Figure -->
                                    <div class="full-text-image-container" id="f6">
                                      <a href="/figure/206311/f6"><img alt="The regulation of S100a6 by transcription factors and SRC-1. (A, B) Relative mRNA levels of S100a6 mRNA measured in the hypothalamus (A) and the hippocampus (B) isolated from control vs. SRC-1-KO mice using Q-PCR. Data are presented as presented as mean ± SEM. N &#x3D; 5 samples per group. *P **P t-tests. (C) Binding sites of transcription factors on the promoter of S100a6 and the effect of indicated transcription factor on S100a6 promoter luciferase activity in Neuro 2A cells. Data are presented as mean ± SEM. N &#x3D; 3–4 repeated experiments with 3 biological replicates per group in each experiment. **P #P D, E) Effects of SRC-1 and transcription factors KLF (D) and AP2 (E) on S100a6 promoter luciferase activity in SRC-1-KO MEF cells. Data are presented as mean ± SEM. N &#x3D; 3–6 repeated experiments with 6 biological replicates per group in each experiment. Control group is normalized to 1 to allow comparisons among different batches of experiments. *P **P ***P " class="full-text-image" src="/article/206311/figure/f6/large"/></a>
                                      <p class="small-font"><span><b>Figure 6.</b></span>  <b>The regulation of S100a6 by transcription factors and SRC-1.</b> (<b>A</b>, <b>B</b>) Relative mRNA levels of S100a6 mRNA measured in the hypothalamus (<b>A</b>) and the hippocampus (<b>B</b>) isolated from control vs. SRC-1-KO mice using Q-PCR. Data are presented as presented as mean ± SEM. <i>N</i> = 5 samples per group. <sup>*</sup><i>P</i> < 0.05, <sup>**</sup><i>P</i> < 0.01 in unpaired two-tailed <i>t</i>-tests. (<b>C</b>) Binding sites of transcription factors on the promoter of S100a6 and the effect of indicated transcription factor on S100a6 promoter luciferase activity in Neuro 2A cells. Data are presented as mean ± SEM. <i>N</i> = 3–4 repeated experiments with 3 biological replicates per group in each experiment. <sup>**</sup><i>P</i> < 0.001 in one-way ANOVA analyses followed by Sidak tests. <sup>#</sup><i>P</i> < 0.05 in unpaired two-tailed t-tests against Control. (<b>D</b>, <b>E</b>) Effects of SRC-1 and transcription factors KLF (<b>D</b>) and AP2 (<b>E</b>) on S100a6 promoter luciferase activity in SRC-1-KO MEF cells. Data are presented as mean ± SEM. <i>N</i> = 3–6 repeated experiments with 6 biological replicates per group in each experiment. Control group is normalized to 1 to allow comparisons among different batches of experiments. <sup>*</sup><i>P</i> < 0.05, <sup>**</sup><i>P</i> < 0.01 and <sup>***</sup><i>P</i> < 0.001 in two-way ANOVA analyses followed by Sidak tests.<br></p>
                                      
                                    </div>
                          
                          
                          
                          
                          
                                <p>To investigate how SRC-1 regulates the expression of S100A6 and S100A11, we analyzed their promoters. Using the online tool PROMO (<a href="http://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3" target="_blank">http://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3</a>), we screened the promoter regions of S100a6 and S100a11 to identify potential transcription factors that may mediate the effects of SRC-1. These findings were further validated using JASPAR transcription factor binding profile database (<a href="https://jaspar.genereg.net/" target="_blank">https://jaspar.genereg.net/</a>). We found several binding sites of five transcription factors, including Sp1, AP2, TCF1, KLF4 and PU.1, on S100a6 and S100a11 gene promoters (<a href="#f6"  class="anchor">Figure 6C</a> and Supplementary <a href="#f5"  class="anchor">Figure 5C</a>). These transcription factors may regulate the processes of AD [<a href="#r55"  class="anchor">55</a>–<a href="#r60"  class="anchor">60</a>]. To confirm their transcriptional activity, we used a Neuro 2A, a mouse neuroblastoma cell line, to perform luciferase assay using the promoters of S100A6 and S100A11. We found that the activity of S100A6 promoter was increased 5 folds by KLF and to a less extend by AP2, while the activity of S100A11 promoter was modestly increased by AP2 and PU.1, but not TCF (<a href="#f6"  class="anchor">Figure 6C</a> and Supplementary <a href="#f5"  class="anchor">Figure 5C</a>).</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                                <p>To further determine whether SRC-1 can enhance the effect of these transcription factors, we performed luciferase assays using SRC-1-KO MEF cells. As expected, both KLF and AP2 significantly increased S100A6 promoter activity. Interestingly, in the background of SRC-1 deficiency, SRC-1 supplement alone did not change S100A6 promoter activity. However, SRC-1 supplement significantly augmented the stimulatory effects of both KLF and AP2 on S100A6 promoter activity (<a href="#f6"  class="anchor">Figure 6D</a>, <a href="#f6"  class="anchor">6E</a>). Similarly, despite that S100A11 promoter activity was not increased by SRC-1 expression, SRC-1 significantly augmented the stimulatory effects of PU.1 and AP2 on S100A11 promoter activity (Supplementary <a href="#f5"  class="anchor">Figure 5D</a>, <a href="#f5"  class="anchor">5E</a>). Together, these results support that the co-activator activity of SRC-1. Although SRC-1 does not regulate the transcription directly, it can enhance the transcriptional activity of AP2 and KLF to promote the expression of S100A6 and enhance the transcriptional activity of PU.1 and AP2 to promote the expression of S100A11.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                </div>
                <div class="section-container" id="discussion">
                  <h2 class="article-header-1"> Discussion</h2>
                  
                        <p>The RNA-Seq analysis of SRC-1-KO brains revealed a negative correlation between SRC-1 expression and neurodegenerative diseases. Consistent with this finding, we identified a protective role of SRC-1 against aging associated cognition decline. With the decline of SRC-1 during aging [<a href="#r22"  class="anchor">22</a>], WT mice displayed a gradual decline of contextual memory. However, SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice displayed lower contextual memory throughout all the life span tested, suggesting an early-aging cognitive phenotype in SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice displayed deficits in contextual memory as early as 6 months of age and persisted the whole life span. Consistently, contextual memory is impaired in some AD mouse models (e.g., Tg2576, 5XFAD and APP/PS1) at 4–6 months of age regardless how soon the Aβ plaques are formed [<a href="#r61"  class="anchor">61</a>–<a href="#r63"  class="anchor">63</a>]. However, 3xTg-AD and AppNL-G-F-knock-in mouse models have intact contextual memory at 6 months of age [<a href="#r64"  class="anchor">64</a>, <a href="#r65"  class="anchor">65</a>]. Interestingly, loss of SRC-1 does not accelerate the development of Aβ plaques in APP/PS1 mice or change the expression of synaptic proteins [<a href="#r19"  class="anchor">19</a>]. These results argue that SRC-1 protects cognitive functions independent of Aβ pathology.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>Despite acute knockdown of SRC-1 in the hippocampus reduces mice’s performance in the Morris water maze test by reducing the LTP machinery [<a href="#r20"  class="anchor">20</a>], our SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice display intact spatial memory during all the tested time in the RAWM test. In addition, while several synaptic proteins are downregulated by acute knockdown of SRC-1 in the hippocampus [<a href="#r20"  class="anchor">20</a>, <a href="#r21"  class="anchor">21</a>], we did not detect the downregulation of these genes in SRC-1-KO mice. It is plausible that these synaptic proteins, but not the mRNAs, are dynamically regulated during memory formation. Another possibility is that the impaired contextual memory in SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice could be independent of the SRC-1 regulatory effects on the hippocampal synaptic proteins. It is also plausible that embryonic deletion or mutation of SRC-1 caused development changes or compensations which rescue the synaptic proteins encoding genes and/or the spatial memory deficits. Thus, SRC-1 may provide different extents of protection or play different roles in different types of memories through different mechanisms.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>To explore the molecular mechanisms, RNA-Seq analysis identified S100 proteins as the most downregulated genes. S100 protein family comprised of at least 25 Ca<sup>2+</sup> or Zn<sup>2+</sup> binding proteins with low molecular weights. Seven S100 proteins that are present in the brain, including S100B, S100A1, S100A6, S100A7, S100A8, S100A9 and S100A12, have been implicated in regulating Aβ levels and Tau phosphorylation [<a href="#r35"  class="anchor">35</a>]. We identified S100A6 and S100A11 as potential downstream target genes of SRC-1, which are associated with the cognition functions and AD pathology [<a href="#r35"  class="anchor">35</a>, <a href="#r38"  class="anchor">38</a>–<a href="#r47"  class="anchor">47</a>]. We also identified the transcription factors that promote the expression of S100A6 and S100A11. Further, we confirmed that SRC-1 can enhance the excitatory effects of KLF and AP2 on S100A6 promoter, and the excitatory effects of PU.1 and AP2 on S100A11 promoter, supporting the co-activator role of SRC-1. Interestingly, the combination of SRC-1 and KLF displayed the most profound stimulation on S100A6 promoter. S100A6 binds Ca<sup>2+</sup> to regulate Ca<sup>2+</sup> homeostasis and Ca<sup>2+</sup>-dependent signaling pathways, and Ca<sup>2+</sup> dysregulation is implicated in AD development [<a href="#r40"  class="anchor">40</a>, <a href="#r41"  class="anchor">41</a>]. Consistently, S100A6 is one of the most significantly positively correlated proteins with the AD phenotype [<a href="#r42"  class="anchor">42</a>]. S100A6 is upregulated in AD patients and in AD mouse models [<a href="#r43"  class="anchor">43</a>–<a href="#r45"  class="anchor">45</a>]. Interestingly, most S100A6 proteins are in astrocytes that surround Aβ plaques [<a href="#r43"  class="anchor">43</a>], and <i>in vitro</i> S100A6 treatment in mouse brain sections reduces Aβ levels and plaque burden [<a href="#r46"  class="anchor">46</a>]. Importantly, S100A6 expression is tightly positively correlated with cognitive function recovery in a rat model of traumatic brain injury [<a href="#r66"  class="anchor">66</a>], implicating the regulatory role of S100A6 in cognition. However, it is unclear whether increased S100A6 causes or defense against these phenotypes.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>Previous studies indicate that acute loss of SRC-1 impairs spatial memory associated with downregulation of hippocampal synaptic proteins encoding genes. Our results support that embryonic loss of SRC-1 impairs contextual memory associated with downregulation of S100A6 and S100A11, independent of the spatial memory or hippocampal synaptic proteins encoding genes. Our studies provide SRC-1 as an upstream regulatory mechanism of S100A6. The synchronous decrease of contextual memory and the downregulation of SRC-1 and S100A6 support a protective role of SRC-1 against aging-associated memory decline, potentially through transcriptional regulation of S100A6.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <p>Aging is associated with both metabolic dysregulations and neurodegenerative diseases, and SRC-1 contributes to both obesity and aging associated dementia. Thus, it is plausible that the decrease of SRC-1 in aging animals contributes to both aging associated body weight gain and cognition loss. Importantly, SRC-1<sup>L1376P</sup> is a mutation identified from human patients with obesity, and SRC-1<sup>L1376P</sup> mice are humanized mutant mouse model, which recapitulate many phenotypes observed in human patients [<a href="#r17"  class="anchor">17</a>]. The SRC-1 coding gene has already been added to the genetic screening panel for obesity, and our study provides a strong rationale to add this gene to the screening panel for neurodegenerative diseases, like AD.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                        <div class="section-container" id="">
                          <h3 class="article-header-2"> Limitation</h3>
                          
                                <p>Our study was conducted only on male mice to avoid the estrous cycle effects. Behavior assays were performed after 6 months of age, which corresponds to middle age in mice. The memory deficits might be developed earlier in SRC-1-KO mice and SRC-1<sup>L1376P</sup> mice. We only explored the regulatory effects of SRC-1 on S100A6 and S100A11 as molecular mechanisms. Many other neurodegenerative diseases-associated molecular changes are worthwhile for future explorations. We used a promoter luciferase assay to evaluate the enhancement effect of SRC-1 on transcription factors. More comprehensive assays, like protein-protein interaction assays, need to be performed to confirm the co-activator function of SRC-1.</p>
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                          
                        </div>
                  
                  
                  
                  
                  
                  
                  
                  
                </div>
                <div class="section-container" id="conclusion">
                  <h2 class="article-header-1"> Conclusion</h2>
                  
                        <p>We identified a negative association between SRC-1 expression and neurodegenerative diseases, and then confirmed the protective effect of SRC-1 against aging associated cognition decline. We further identified the stimulatory effect of SRC-1 on the transcription of S100A6 and S100A11, a potential molecular mechanism.</p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                </div>
                <div class="section-container" id="supplementarymaterials">
                  <h2 class="article-header-1"> Supplementary Materials</h2>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                              <h3 class="article-header-3"><a href="/article/206311/supplementary/SD1/0/aging-v17i8-206311-supplementary-material-SD1.pdf" class="supplemental" id="SD1"><span class="icon icon-download"></span>
                                    Supplementary Figures
                                </a></h3>
                  
                          <p></p>
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                  
                              <h3 class="article-header-3"><a href="/article/206311/supplementary/SD2/0/aging-v17i8-206311-supplementary-material-SD2.xlsx" class="supplemental" id="SD2"><span class="icon icon-download"></span>
                                    Supplementary Data 1
                                </a></h3>
                  
                          <p></p>
                  
                  
                </div>


                  <h2 id="authorcontributions" class="article-header-1">Author Contributions</h2>
                    <p>Hesong Liu performed all behavior experiments. Y.Y performed all molecular studies. J.B performed RNA-Seq analysis and spatial RNA-Seq secondary analysis. Y.H performed electrophysiology experiments. R.M performed bulk RNA-Seq secondary analysis. H.L (Hailan Liu) performed immunofluorescence experiments. C.L, N.Z, M.Y, L.T, Q.L, Y.D, K.C, N.Y, M.W, Y.L, J.H, S.J and M.B assisted in experiments and data collections. H.K.Y and C.W conceived, designed, and supervised the project. All the experiments were performed at Children’s Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine.</p>




                  <h2 id="conflictsofinterest" class="article-header-1">Conflicts of Interest</h2>
                  
                    <p>The authors declare no conflicts of interest related to this study.</p>
                  <h2 id="ethicalstatement" class="article-header-1">Ethical Statement</h2>
                  
                    <p>Care of all animals and procedures were approved by the Institutional Animal Care and Use Committee of Baylor College of Medicine (AN-5479 and AN-6098). Human data are from publicly available GTEx data, and these datasets are de-identified and do not contain information that can be linked to individual donors, aligning with ethical guidelines that exempt such data from human subject’s research regulations.</p>
                  <h2 id="funding" class="article-header-1">Funding</h2>
                  
                    <p>Investigators involved in this work were supported by grants from the NIH R01 DK136627 to C.W., USDA/ARS under Cooperative Agreement No. 58-3092-0-001, USDA/ARS grant CRIS 3092-51000-065-003S, the Research Vision at Texas Children's Hospital, and the Duncan NRI Zoghbi Scholar Award to H.K.Y. American Heart Association award 23POST1030352 to Hailan Liu.</p>

              <h2 id="references">References</h2>
              <ul class="references">
                  <li id="r1">
                    <b>1.</b>

Hebert LE, Weuve J, Scherr PA, Evans DA. Alzheimer disease in the United States (2010-2050) estimated using the 2010 census. Neurology. 2013; 80:1778–83.  <a href="https://doi.org/10.1212/WNL.0b013e31828726f5">https://doi.org/10.1212/WNL.0b013e31828726f5</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23390181" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r2">
                    <b>2.</b>

Flores-Cordero JA, Pérez-Pérez A, Jiménez-Cortegana C, Alba G, Flores-Barragán A, Sánchez-Margalet V. Obesity as a Risk Factor for Dementia and Alzheimer's Disease: The Role of Leptin. Int J Mol Sci. 2022; 23:5202.  <a href="https://doi.org/10.3390/ijms23095202">https://doi.org/10.3390/ijms23095202</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/35563589" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r3">
                    <b>3.</b>

Morys F, Potvin O, Zeighami Y, Vogel J, Lamontagne-Caron R, Duchesne S, Dagher A, and Alzheimer’s Disease Neuroimaging Initiative. Obesity-Associated Neurodegeneration Pattern Mimics Alzheimer's Disease in an Observational Cohort Study. J Alzheimers Dis. 2023; 91:1059–71.  <a href="https://doi.org/10.3233/JAD-220535">https://doi.org/10.3233/JAD-220535</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/36565111" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r4">
                    <b>4.</b>

Suzzi S, Croese T, Ravid A, Gold O, Clark AR, Medina S, Kitsberg D, Adam M, Vernon KA, Kohnert E, Shapira I, Malitsky S, Itkin M, et al. N-acetylneuraminic acid links immune exhaustion and accelerated memory deficit in diet-induced obese Alzheimer's disease mouse model. Nat Commun. 2023; 14:1293.  <a href="https://doi.org/10.1038/s41467-023-36759-8">https://doi.org/10.1038/s41467-023-36759-8</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/36894557" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r5">
                    <b>5.</b>

Ly M, Yu GZ, Mian A, Cramer A, Meysami S, Merrill DA, Samara A, Eisenstein SA, Hershey T, Babulal GM, Lenze EJ, Morris JC, Benzinger TLS, Raji CA. Neuroinflammation: A Modifiable Pathway Linking Obesity, Alzheimer's disease, and Depression. Am J Geriatr Psychiatry. 2023; 31:853–66.  <a href="https://doi.org/10.1016/j.jagp.2023.06.001">https://doi.org/10.1016/j.jagp.2023.06.001</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/37365110" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r6">
                    <b>6.</b>

Picone P, Di Carlo M, Nuzzo D. Obesity and Alzheimer's disease: Molecular bases. Eur J Neurosci. 2020; 52:3944–50.  <a href="https://doi.org/10.1111/ejn.14758">https://doi.org/10.1111/ejn.14758</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32323378" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r7">
                    <b>7.</b>

Wang Y, Fisahn A, Sinha I, Nguyen DP, Sterzenbach U, Lallemend F, Hadjab S. Hippocampal Transcriptome Profile of Persistent Memory Rescue in a Mouse Model of THRA1 Mutation-Mediated Resistance to Thyroid Hormone. Sci Rep. 2016; 6:18617.  <a href="https://doi.org/10.1038/srep18617">https://doi.org/10.1038/srep18617</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26743578" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r8">
                    <b>8.</b>

Tertil M, Skupio U, Barut J, Dubovyk V, Wawrzczak-Bargiela A, Soltys Z, Golda S, Kudla L, Wiktorowska L, Szklarczyk K, Korostynski M, Przewlocki R, Slezak M. Glucocorticoid receptor signaling in astrocytes is required for aversive memory formation. Transl Psychiatry. 2018; 8:255.  <a href="https://doi.org/10.1038/s41398-018-0300-x">https://doi.org/10.1038/s41398-018-0300-x</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/30487639" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r9">
                    <b>9.</b>

Kim J, Frick KM. Distinct effects of estrogen receptor antagonism on object recognition and spatial memory consolidation in ovariectomized mice. Psychoneuroendocrinology. 2017; 85:110–4.  <a href="https://doi.org/10.1016/j.psyneuen.2017.08.013">https://doi.org/10.1016/j.psyneuen.2017.08.013</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/28846921" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r10">
                    <b>10.</b>

Govindarajulu M, Pinky PD, Bloemer J, Ghanei N, Suppiramaniam V, Amin R. Signaling Mechanisms of Selective PPAR<i>γ</i> Modulators in Alzheimer's Disease. PPAR Res. 2018; 2018:2010675.  <a href="https://doi.org/10.1155/2018/2010675">https://doi.org/10.1155/2018/2010675</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/30420872" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r11">
                    <b>11.</b>

Reichenbach N, Delekate A, Plescher M, Schmitt F, Krauss S, Blank N, Halle A, Petzold GC. Inhibition of Stat3-mediated astrogliosis ameliorates pathology in an Alzheimer's disease model. EMBO Mol Med. 2019; 11:e9665.  <a href="https://doi.org/10.15252/emmm.201809665">https://doi.org/10.15252/emmm.201809665</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/30617153" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r12">
                    <b>12.</b>

Fu JB, Wang ZH, Ren YY. Forkhead Box O1-p21 Mediates Macrophage Polarization in Postoperative Cognitive Dysfunction Induced by Sevoflurane. Curr Neurovasc Res. 2020; 17:79–85.  <a href="https://doi.org/10.2174/1567202617666200128142728">https://doi.org/10.2174/1567202617666200128142728</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32003673" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r13">
                    <b>13.</b>

York B, O'Malley BW. Steroid receptor coactivator (SRC) family: masters of systems biology. J Biol Chem. 2010; 285:38743–50.  <a href="https://doi.org/10.1074/jbc.R110.193367">https://doi.org/10.1074/jbc.R110.193367</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/20956538" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r14">
                    <b>14.</b>

Meijer OC, Steenbergen PJ, De Kloet ER. Differential expression and regional distribution of steroid receptor coactivators SRC-1 and SRC-2 in brain and pituitary. Endocrinology. 2000; 141:2192–9.  <a href="https://doi.org/10.1210/endo.141.6.7489">https://doi.org/10.1210/endo.141.6.7489</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10830308" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r15">
                    <b>15.</b>

Zhu L, Yang Y, Xu P, Zou F, Yan X, Liao L, Xu J, O'Malley BW, Xu Y. Steroid receptor coactivator-1 mediates estrogenic actions to prevent body weight gain in female mice. Endocrinology. 2013; 154:150–8.  <a href="https://doi.org/10.1210/en.2012-2007">https://doi.org/10.1210/en.2012-2007</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23211707" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r16">
                    <b>16.</b>

Picard F, Géhin M, Annicotte J, Rocchi S, Champy MF, O'Malley BW, Chambon P, Auwerx J. SRC-1 and TIF2 control energy balance between white and brown adipose tissues. Cell. 2002; 111:931–41.  <a href="https://doi.org/10.1016/s0092-8674(02)01169-8">https://doi.org/10.1016/s0092-8674(02)01169-8</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12507421" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r17">
                    <b>17.</b>

Yang Y, van der Klaauw AA, Zhu L, Cacciottolo TM, He Y, Stadler LKJ, Wang C, Xu P, Saito K, Hinton A<suffix> Jr</suffix>, Yan X, Keogh JM, Henning E, et al, and UK10K Consortium. Steroid receptor coactivator-1 modulates the function of Pomc neurons and energy homeostasis. Nat Commun. 2019; 10:1718.  <a href="https://doi.org/10.1038/s41467-019-08737-6">https://doi.org/10.1038/s41467-019-08737-6</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/30979869" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r18">
                    <b>18.</b>

Pharmaceuticals R. Rhythm Pharmaceuticals Announces Positive Data with Setmelanotide in Additional MC4R Pathway Deficiency-related Obesities. Globe Newswire. 2021. 

                  </li>
                  <li id="r19">
                    <b>19.</b>

Wu Q, Wang B, Li QF, Zhang X, Ntim M, Wu XF, Li N, Zhu DD, Jiang R, Yang JY, Yuan YH, Li S. SRC-1 Knockout Exerts No Effect on Amyloid β Deposition in APP/PS1 Mice. Front Aging Neurosci. 2020; 12:145.  <a href="https://doi.org/10.3389/fnagi.2020.00145">https://doi.org/10.3389/fnagi.2020.00145</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32625077" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r20">
                    <b>20.</b>

Bian C, Huang Y, Zhu H, Zhao Y, Zhao J, Zhang J. Steroid Receptor Coactivator-1 Knockdown Decreases Synaptic Plasticity and Impairs Spatial Memory in the Hippocampus of Mice. Neuroscience. 2018; 377:114–25.  <a href="https://doi.org/10.1016/j.neuroscience.2018.02.034">https://doi.org/10.1016/j.neuroscience.2018.02.034</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/29524638" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r21">
                    <b>21.</b>

Chen X, Tian Y, Zhu H, Bian C, Li M. Inhibition of steroid receptor coactivator-1 in the hippocampus impairs the consolidation and reconsolidation of contextual fear memory in mice. Life Sci. 2020; 245:117386.  <a href="https://doi.org/10.1016/j.lfs.2020.117386">https://doi.org/10.1016/j.lfs.2020.117386</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32006528" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r22">
                    <b>22.</b>

Zhang D, Guo Q, Bian C, Zhang J, Lin S, Su B. Alterations of steroid receptor coactivator-1 (SRC-1) immunoreactivities in specific brain regions of young and middle-aged female Sprague-Dawley rats. Brain Res. 2011; 1382:88–97.  <a href="https://doi.org/10.1016/j.brainres.2011.01.024">https://doi.org/10.1016/j.brainres.2011.01.024</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21241680" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r23">
                    <b>23.</b>

Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, Batut P, Chaisson M, Gingeras TR. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013; 29:15–21.  <a href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r24">
                    <b>24.</b>

Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014; 30:923–30.  <a href="https://doi.org/10.1093/bioinformatics/btt656">https://doi.org/10.1093/bioinformatics/btt656</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24227677" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r25">
                    <b>25.</b>

Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014; 15:550.  <a href="https://doi.org/10.1186/s13059-014-0550-8">https://doi.org/10.1186/s13059-014-0550-8</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r26">
                    <b>26.</b>

Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES, Mesirov JP. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A. 2005; 102:15545–50.  <a href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16199517" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r27">
                    <b>27.</b>

Shen S, Park JW, Lu ZX, Lin L, Henry MD, Wu YN, Zhou Q, Xing Y. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Proc Natl Acad Sci U S A. 2014; 111:E5593–601.  <a href="https://doi.org/10.1073/pnas.1419161111">https://doi.org/10.1073/pnas.1419161111</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25480548" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r28">
                    <b>28.</b>

Chen WT, Lu A, Craessaerts K, Pavie B, Sala Frigerio C, Corthout N, Qian X, Laláková J, Kühnemund M, Voytyuk I, Wolfs L, Mancuso R, Salta E, et al. Spatial Transcriptomics and In Situ Sequencing to Study Alzheimer's Disease. Cell. 2020; 182:976–91.e19.  <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32702314" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r29">
                    <b>29.</b>

Butler A, Hoffman P, Smibert P, Papalexi E, Satija R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol. 2018; 36:411–20.  <a href="https://doi.org/10.1038/nbt.4096">https://doi.org/10.1038/nbt.4096</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/29608179" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r30">
                    <b>30.</b>

Liu H, He Y, Liu H, Brouwers B, Yin N, Lawler K, Keogh JM, Henning E, Lee DK, Yu M, Tu L, Zhang N, Conde KM, et al. Neural circuits expressing the serotonin 2C receptor regulate memory in mice and humans. Sci Adv. 2024; 10:eadl2675.  <a href="https://doi.org/10.1126/sciadv.adl2675">https://doi.org/10.1126/sciadv.adl2675</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/38941473" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r31">
                    <b>31.</b>

Leger M, Quiedeville A, Bouet V, Haelewyn B, Boulouard M, Schumann-Bard P, Freret T. Object recognition test in mice. Nat Protoc. 2013; 8:2531–7.  <a href="https://doi.org/10.1038/nprot.2013.155">https://doi.org/10.1038/nprot.2013.155</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24263092" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r32">
                    <b>32.</b>

Lin L, Pang W, Chen K, Wang F, Gengler J, Sun Y, Tong Q. Adipocyte expression of PU.1 transcription factor causes insulin resistance through upregulation of inflammatory cytokine gene expression and ROS production. Am J Physiol Endocrinol Metab. 2012; 302:E1550–9.  <a href="https://doi.org/10.1152/ajpendo.00462.2011">https://doi.org/10.1152/ajpendo.00462.2011</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22454293" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r33">
                    <b>33.</b>

Sheng JG, Mrak RE, Griffin WS. S100 beta protein expression in Alzheimer disease: potential role in the pathogenesis of neuritic plaques. J Neurosci Res. 1994; 39:398–404.  <a href="https://doi.org/10.1002/jnr.490390406">https://doi.org/10.1002/jnr.490390406</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7884819" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r34">
                    <b>34.</b>

Van Eldik LJ, Griffin WS. S100 beta expression in Alzheimer's disease: relation to neuropathology in brain regions. Biochim Biophys Acta. 1994; 1223:398–403.  <a href="https://doi.org/10.1016/0167-4889(94)90101-5">https://doi.org/10.1016/0167-4889(94)90101-5</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7918676" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r35">
                    <b>35.</b>

Cristóvão JS, Gomes CM. S100 Proteins in Alzheimer's Disease. Front Neurosci. 2019; 13:463.  <a href="https://doi.org/10.3389/fnins.2019.00463">https://doi.org/10.3389/fnins.2019.00463</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31156365" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r36">
                    <b>36.</b>

Depp C, Sun T, Sasmita AO, Spieth L, Berghoff SA, Nazarenko T, Overhoff K, Steixner-Kumar AA, Subramanian S, Arinrad S, Ruhwedel T, Möbius W, Göbbels S, et al. Myelin dysfunction drives amyloid-β deposition in models of Alzheimer's disease. Nature. 2023; 618:349–57.  <a href="https://doi.org/10.1038/s41586-023-06120-6">https://doi.org/10.1038/s41586-023-06120-6</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/37258678" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r37">
                    <b>37.</b>

Nasrabady SE, Rizvi B, Goldman JE, Brickman AM. White matter changes in Alzheimer's disease: a focus on myelin and oligodendrocytes. Acta Neuropathol Commun. 2018; 6:22.  <a href="https://doi.org/10.1186/s40478-018-0515-3">https://doi.org/10.1186/s40478-018-0515-3</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/29499767" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r38">
                    <b>38.</b>

Xia Q, Li X, Zhou H, Zheng L, Shi J. S100A11 protects against neuronal cell apoptosis induced by cerebral ischemia via inhibiting the nuclear translocation of annexin A1. Cell Death Dis. 2018; 9:657.  <a href="https://doi.org/10.1038/s41419-018-0686-7">https://doi.org/10.1038/s41419-018-0686-7</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/29844306" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r39">
                    <b>39.</b>

Ramos-Campoy O, Lladó A, Bosch B, Ferrer M, Pérez-Millan A, Vergara M, Molina-Porcel L, Fort-Aznar L, Gonzalo R, Moreno-Izco F, Fernandez-Villullas G, Balasa M, Sánchez-Valle R, Antonell A. Differential Gene Expression in Sporadic and Genetic Forms of Alzheimer's Disease and Frontotemporal Dementia in Brain Tissue and Lymphoblastoid Cell Lines. Mol Neurobiol. 2022; 59:6411–28.  <a href="https://doi.org/10.1007/s12035-022-02969-2">https://doi.org/10.1007/s12035-022-02969-2</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/35962298" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r40">
                    <b>40.</b>

Bojarski L, Herms J, Kuznicki J. Calcium dysregulation in Alzheimer's disease. Neurochem Int. 2008; 52:621–33.  <a href="https://doi.org/10.1016/j.neuint.2007.10.002">https://doi.org/10.1016/j.neuint.2007.10.002</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18035450" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r41">
                    <b>41.</b>

Haldar B, Hamilton CL, Solodushko V, Abney KA, Alexeyev M, Honkanen RE, Scammell JG, Cioffi DL. S100A6 is a positive regulator of PPP5C-FKBP51-dependent regulation of endothelial calcium signaling. FASEB J. 2020; 34:3179–96.  <a href="https://doi.org/10.1096/fj.201901777R">https://doi.org/10.1096/fj.201901777R</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31916625" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r42">
                    <b>42.</b>

Wruck W, Schröter F, Adjaye J. Meta-Analysis of Transcriptome Data Related to Hippocampus Biopsies and iPSC-Derived Neuronal Cells from Alzheimer's Disease Patients Reveals an Association with FOXA1 and FOXA2 Gene Regulatory Networks. J Alzheimers Dis. 2016; 50:1065–82.  <a href="https://doi.org/10.3233/JAD-150733">https://doi.org/10.3233/JAD-150733</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26890743" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r43">
                    <b>43.</b>

Boom A, Pochet R, Authelet M, Pradier L, Borghgraef P, Van Leuven F, Heizmann CW, Brion JP. Astrocytic calcium/zinc binding protein S100A6 over expression in Alzheimer's disease and in PS1/APP transgenic mice models. Biochim Biophys Acta. 2004; 1742:161–8.  <a href="https://doi.org/10.1016/j.bbamcr.2004.09.011">https://doi.org/10.1016/j.bbamcr.2004.09.011</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15590066" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r44">
                    <b>44.</b>

Wirths O, Breyhan H, Marcello A, Cotel MC, Brück W, Bayer TA. Inflammatory changes are tightly associated with neurodegeneration in the brain and spinal cord of the APP/PS1KI mouse model of Alzheimer's disease. Neurobiol Aging. 2010; 31:747–57.  <a href="https://doi.org/10.1016/j.neurobiolaging.2008.06.011">https://doi.org/10.1016/j.neurobiolaging.2008.06.011</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18657882" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r45">
                    <b>45.</b>

Weissmann R, Hüttenrauch M, Kacprowski T, Bouter Y, Pradier L, Bayer TA, Kuss AW, Wirths O. Gene Expression Profiling in the APP/PS1KI Mouse Model of Familial Alzheimer's Disease. J Alzheimers Dis. 2016; 50:397–409.  <a href="https://doi.org/10.3233/JAD-150745">https://doi.org/10.3233/JAD-150745</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26639971" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r46">
                    <b>46.</b>

Tian ZY, Wang CY, Wang T, Li YC, Wang ZY. Glial S100A6 Degrades β-amyloid Aggregation through Targeting Competition with Zinc Ions. Aging Dis. 2019; 10:756–69.  <a href="https://doi.org/10.14336/AD.2018.0912">https://doi.org/10.14336/AD.2018.0912</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31440382" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r47">
                    <b>47.</b>

Filipek A, Leśniak W. S100A6 and Its Brain Ligands in Neurodegenerative Disorders. Int J Mol Sci. 2020; 21:3979.  <a href="https://doi.org/10.3390/ijms21113979">https://doi.org/10.3390/ijms21113979</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32492924" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r48">
                    <b>48.</b>

Nuzzo D, Inguglia L, Walters J, Picone P, Di Carlo M. A Shotgun Proteomics Approach Reveals a New Toxic Role for Alzheimer's Disease Aβ Peptide: Spliceosome Impairment. J Proteome Res. 2017; 16:1526–41.  <a href="https://doi.org/10.1021/acs.jproteome.6b00925">https://doi.org/10.1021/acs.jproteome.6b00925</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/28157316" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r49">
                    <b>49.</b>

Hsieh YC, Guo C, Yalamanchili HK, Abreha M, Al-Ouran R, Li Y, Dammer EB, Lah JJ, Levey AI, Bennett DA, De Jager PL, Seyfried NT, Liu Z, Shulman JM. Tau-Mediated Disruption of the Spliceosome Triggers Cryptic RNA Splicing and Neurodegeneration in Alzheimer's Disease. Cell Rep. 2019; 29:301–16.e10.  <a href="https://doi.org/10.1016/j.celrep.2019.08.104">https://doi.org/10.1016/j.celrep.2019.08.104</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31597093" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r50">
                    <b>50.</b>

Biamonti G, Amato A, Belloni E, Di Matteo A, Infantino L, Pradella D, Ghigna C. Alternative splicing in Alzheimer's disease. Aging Clin Exp Res. 2021; 33:747–58.  <a href="https://doi.org/10.1007/s40520-019-01360-x">https://doi.org/10.1007/s40520-019-01360-x</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31583531" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r51">
                    <b>51.</b>

Cohen SJ, Stackman RW<suffix> Jr</suffix>. Assessing rodent hippocampal involvement in the novel object recognition task. A review. Behav Brain Res. 2015; 285:105–17.  <a href="https://doi.org/10.1016/j.bbr.2014.08.002">https://doi.org/10.1016/j.bbr.2014.08.002</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25169255" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r52">
                    <b>52.</b>

Cinalli DA<suffix> Jr</suffix>, Cohen SJ, Guthrie K, Stackman RW<suffix> Jr</suffix>. Object Recognition Memory: Distinct Yet Complementary Roles of the Mouse CA1 and Perirhinal Cortex. Front Mol Neurosci. 2020; 13:527543.  <a href="https://doi.org/10.3389/fnmol.2020.527543">https://doi.org/10.3389/fnmol.2020.527543</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/33192287" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r53">
                    <b>53.</b>

Xu C, Krabbe S, Gründemann J, Botta P, Fadok JP, Osakada F, Saur D, Grewe BF, Schnitzer MJ, Callaway EM, Lüthi A. Distinct Hippocampal Pathways Mediate Dissociable Roles of Context in Memory Retrieval. Cell. 2016; 167:961–72.e16.  <a href="https://doi.org/10.1016/j.cell.2016.09.051">https://doi.org/10.1016/j.cell.2016.09.051</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27773481" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r54">
                    <b>54.</b>

Kim WB, Cho JH. Encoding of contextual fear memory in hippocampal-amygdala circuit. Nat Commun. 2020; 11:1382.  <a href="https://doi.org/10.1038/s41467-020-15121-2">https://doi.org/10.1038/s41467-020-15121-2</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/32170133" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r55">
                    <b>55.</b>

Al-Sabri MH, Nikpour M, Clemensson LE, Attwood MM, Williams MJ, Rask-Anderson M, Mwinyi J, Schiöth HB. The regulatory role of AP-2β in monoaminergic neurotransmitter systems: insights on its signalling pathway, linked disorders and theragnostic potential. Cell Biosci. 2022; 12:151.  <a href="https://doi.org/10.1186/s13578-022-00891-7">https://doi.org/10.1186/s13578-022-00891-7</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/36076256" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r56">
                    <b>56.</b>

Cheng Z, Zou X, Jin Y, Gao S, Lv J, Li B, Cui R. The Role of KLF<sub>4</sub> in Alzheimer's Disease. Front Cell Neurosci. 2018; 12:325.  <a href="https://doi.org/10.3389/fncel.2018.00325">https://doi.org/10.3389/fncel.2018.00325</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/30297986" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r57">
                    <b>57.</b>

Jia L, Piña-Crespo J, Li Y. Restoring Wnt/β-catenin signaling is a promising therapeutic strategy for Alzheimer's disease. Mol Brain. 2019; 12:104.  <a href="https://doi.org/10.1186/s13041-019-0525-5">https://doi.org/10.1186/s13041-019-0525-5</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/31801553" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r58">
                    <b>58.</b>

Kim B, Dabin LC, Tate MD, Karahan H, Sharify AD, Acri DJ, Al-Amin MM, Philtjens S, Smith DC, Wijeratne HRS, Park JH, Jucker M, Kim J. Effects of SPI1-mediated transcriptome remodeling on Alzheimer's disease-related phenotypes in mouse models of Aβ amyloidosis. Nat Commun. 2024; 15:3996.  <a href="https://doi.org/10.1038/s41467-024-48484-x">https://doi.org/10.1038/s41467-024-48484-x</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/38734693" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r59">
                    <b>59.</b>

Liu J, Xiao Q, Xiao J, Niu C, Li Y, Zhang X, Zhou Z, Shu G, Yin G. Wnt/β-catenin signalling: function, biological mechanisms, and therapeutic opportunities. Signal Transduct Target Ther. 2022; 7:3.  <a href="https://doi.org/10.1038/s41392-021-00762-6">https://doi.org/10.1038/s41392-021-00762-6</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/34980884" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r60">
                    <b>60.</b>

Villa C, Ridolfi E, Fenoglio C, Ghezzi L, Vimercati R, Clerici F, Marcone A, Gallone S, Serpente M, Cantoni C, Bonsi R, Cioffi S, Cappa S, et al. Expression of the transcription factor Sp1 and its regulatory hsa-miR-29b in peripheral blood mononuclear cells from patients with Alzheimer's disease. J Alzheimers Dis. 2013; 35:487–94.  <a href="https://doi.org/10.3233/JAD-122263">https://doi.org/10.3233/JAD-122263</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23435408" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r61">
                    <b>61.</b>

Ohno M. Failures to reconsolidate memory in a mouse model of Alzheimer's disease. Neurobiol Learn Mem. 2009; 92:455–9.  <a href="https://doi.org/10.1016/j.nlm.2009.05.001">https://doi.org/10.1016/j.nlm.2009.05.001</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19435612" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r62">
                    <b>62.</b>

Dodge JC, Tamsett TJ, Treleaven CM, Taksir TV, Piepenhagen P, Sardi SP, Cheng SH, Shihabuddin LS. Glucosylceramide synthase inhibition reduces ganglioside GM3 accumulation, alleviates amyloid neuropathology, and stabilizes remote contextual memory in a mouse model of Alzheimer's disease. Alzheimers Res Ther. 2022; 14:19.  <a href="https://doi.org/10.1186/s13195-022-00966-0">https://doi.org/10.1186/s13195-022-00966-0</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/35105352" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r63">
                    <b>63.</b>

Gong B, Cao Z, Zheng P, Vitolo OV, Liu S, Staniszewski A, Moolman D, Zhang H, Shelanski M, Arancio O. Ubiquitin hydrolase Uch-L1 rescues beta-amyloid-induced decreases in synaptic function and contextual memory. Cell. 2006; 126:775–88.  <a href="https://doi.org/10.1016/j.cell.2006.06.046">https://doi.org/10.1016/j.cell.2006.06.046</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16923396" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r64">
                    <b>64.</b>

Stover KR, Campbell MA, Van Winssen CM, Brown RE. Early detection of cognitive deficits in the 3xTg-AD mouse model of Alzheimer's disease. Behav Brain Res. 2015; 289:29–38.  <a href="https://doi.org/10.1016/j.bbr.2015.04.012">https://doi.org/10.1016/j.bbr.2015.04.012</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25896362" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r65">
                    <b>65.</b>

Bellio TA, Laguna-Torres JY, Campion MS, Chou J, Yee S, Blusztajn JK, Mellott TJ. Perinatal choline supplementation prevents learning and memory deficits and reduces brain amyloid Aβ42 deposition in AppNL-G-F Alzheimer's disease model mice. PLoS One. 2024; 19:e0297289.  <a href="https://doi.org/10.1371/journal.pone.0297289">https://doi.org/10.1371/journal.pone.0297289</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/38315685" target="_BLANK">PubMed</a>]</span>

                  </li>
                  <li id="r66">
                    <b>66.</b>

Fang B, Liang M, Yang G, Ye Y, Xu H, He X, Huang JH. Expression of S100A6 in rat hippocampus after traumatic brain injury due to lateral head acceleration. Int J Mol Sci. 2014; 15:6378–90.  <a href="https://doi.org/10.3390/ijms15046378">https://doi.org/10.3390/ijms15046378</a><span> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24739809" target="_BLANK">PubMed</a>]</span>

                  </li>
              </ul>
          </div>
          

        <div id="trendmd-suggestions" style="display:none;"></div>

  </div>

    <div class="page-sidebar col s12 m12 l4">
              <a href="/article/206311/pdf" class="download-pdf full-width end-cap primary button">
                <span class="icon icon-download"></span> Download PDF
              </a>
    
    
    
    
    
          <div class="card sidebar-card">
            <h6 class="card-title">Corresponding Authors</h6>
              <p class="correspondence">
                <span class="correspondence-text">Hari Krishna Yalamanchili</span><br /> 
                  <a class="correspondence-email" href="mailto:hari.yalamanchili@bcm.edu">hari.yalamanchili@bcm.edu</a><br />
                
              </p>
              <p class="correspondence">
                <span class="correspondence-text">Chunmei Wang</span><br /> 
                  <a class="correspondence-email" href="mailto:chunmeiw@bcm.edu">chunmeiw@bcm.edu</a><br />
                
              </p>
          </div>
    
    
          <div class="card sidebar-card">
            <h6 class="card-title">Keywords</h6>
            <div class="tags">
                <a class="keyword tag" href="/search?keywords=aging&searchPage=1">aging</a>
                <a class="keyword tag" href="/search?keywords=dementia&searchPage=1">dementia</a>
                <a class="keyword tag" href="/search?keywords=SRC-1&searchPage=1">SRC-1</a>
                <a class="keyword tag" href="/search?keywords=S100A6&searchPage=1">S100A6</a>
            </div>
          </div>
          
          
    
            <div class="sidebar-card fixed-scroll-card">
              <div class="card">
                <h6 class="card-title">Paper Sections</h6>
                <ul class="scrollspy toc">
                    <li><a class="anchor" href="#abstract">Abstract</a></li>
                  
                      <li><a class="anchor" href="#introduction">Introduction</a></li>
                      <li><a class="anchor" href="#methods">Methods</a></li>
                      <li><a class="anchor" href="#results">Results</a></li>
                      <li><a class="anchor" href="#discussion">Discussion</a></li>
                      <li><a class="anchor" href="#conclusion">Conclusion</a></li>
                      <li><a class="anchor" href="#supplementarymaterials">Supplementary Materials</a></li>
                      <li><a class="anchor" href="#authorcontributions">Author Contributions</a></li>
                      <li><a class="anchor" href="#conflictsofinterest">Conflicts of Interest</a></li>
                      <li><a class="anchor" href="#ethicalstatement">Ethical Statement</a></li>
                      <li><a class="anchor" href="#funding">Funding</a></li>
                      <li><a class="anchor" href="#references">References</a></li>
                  
                </ul>
              </div>
            </div>
    </div>
</div>




        </div>
      </div>


    </div>
    </div>

    <div >
        <footer>
          <div class="footer-list">
            <p><a href="/">Home</a> | <a href="/editorial-board">Editorial Board</a> | <a href="/editorial-policies">Editorial Policies</a> | <a href="/advance-publications">Advance Publications</a> | <a href="/current">Current Issue</a> | <a href="/archive">Archive</a> | <a href="/special-collections-archive">Special Collections</a> | <a href="/scientific-integrity">Scientific Integrity</a> | <a href="/ethics">Publication Ethics Statements</a> | <a href="/interviews">Interviews with Outstanding Authors</a> | <a href="/news_room/press_release">Newsroom</a> | <a href="/news-room/conferences">Sponsored Conferences</a> | <a href="/contact">Contact</a></p><p>
    </p>
          </div>
    
            <div class="cookies-policy-box">
              <p>By using our site you are giving us permission to use cookies. This website collects cookies to deliver a better user experience, and to analyze our website traffic and performance. Personal data is not collected. <a href="https://www.aging-us.com/privacy-policy" rel="noopener noreferrer" target="_blank">Privacy Policy</a> | <a href="https://www.aging-us.com/terms-of-service" rel="noopener noreferrer" target="_blank">Terms Of Service</a></p>
            </div>
    
            <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank"><img src="/images/CCBY-80x15.png" alt="Creative Commons Attribution 4.0 License (CC BY 4.0)" style="vertical-align:text-bottom;"></a> <span>All site content, except where otherwise noted, is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution 4.0 License (CC BY 4.0)</a>.</span>
    
        </footer>
    </div>
    
      <script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"76719", "element":"#trendmd-suggestions"}'></script>
  </body>
</html>
